## Introduction

Being sessile organisms, plants have to time and regulate their development based on seasonal and environmental cues.
One of the seasonal cues that plants are capable of responding to is the prolonged cold of winter[@song_remembering_2013].
In the model species Arabidopsis, accessions are either summer or winter annuals[@shindo_natural_2007; @koornneef_naturally_2004].
Summer annuals germinate and set seed in the same year by germinating in the spring, flowering in the summer, and setting seed before the winter months.
Conversely, winter plants germinate in the autumn, stay vegetative over the winter months, then flower and set seed the following spring or summer.
Without experiencing an extended period of cold, winter annual plants may not flower or flowering may be severely delayed.
Delaying the floral transition until an extended period of cold is experienced is a vernalization response; an evolutionary adaptation to the climate where the plants are growing[@thompson_spatiotemporal_1994].
One of the central genes in the vernalization response is *FLC*, a MADS-box containing transcription factor[@michaels_flowering_1999].
The discovery of *FLC* is closely linked with the discovery of the gene *FRI*.
Comparing an early flowering accession and a late flowering, vernalization-responsive accession of Arabidopsis revealed an active *FRI* allele as being required for the latter phenotype[@lee_analysis_1993].
However, when this active *FRI* allele was crossed into another Arabidopsis accession that does not require vernalization, the inheritance of the late flowering phenotype indicated that a locus in addition to *FRI* was required for the late flowering phenotype[@lee_late_flowering_1994; @clarke_mapping_1994; @koornneef_phenotype_1994].
Through additional studies it was determined that a winter annual life strategy was largely conferred through active alleles of both *FRI* and *FLC*.
Sequence polymorphisms in the first intron of *FLC* conferred a summer annual growth habit on some Arabidopsis accessions[@michaels_attenuation_2003], while different *FLC* alleles were found to alter the length of vernalization required to accelerate flowering[@shindo_variation_2006].
A Swedish variety of Arabidopsis, Lov-1, was found to require a longer period of vernalization to fully repress *FLC* expression relative to other accessions[@coustham_quantitative_2012].
The *FLC* allele from the Lov-1 accession has a higher optimum vernalization temperature than other tested accessions, and this is proposed to be an adaptation to the snowfall experienced by the plants in their natural region of growth in northern Sweden[@duncan_seasonal_2015].
Although *FLC* is important for the vernalization response, sequence variation at *FRI* was responsible for ~70% of flowering time variation in a collection of natural Arabidopsis accessions[@shindo_role_2005].
This result highlights the importance of both genes for conferring a winter growth habit in Arabidopsis.

*FLC* is a floral inhibitor[@michaels_flowering_1999] controlled by both the autonomous and vernalization flowering time pathways, that binds to and represses the expression of *FT*[@helliwell_arabidopsis_2006] in addition to other floral integrators[@searle_transcription_2006].
The autonomous pathway increases the expression of *FLC* while the vernalization pathway represses expression of the gene[@sheldon_flf_1999; @koornneef_genetic_1998; @sanda_interaction_1996; @michaels_loss_2001].
The expression of the gene was found to decrease during vernalization in a quantitative manner, with the more cold the plant experienced, the more the gene was repressed[@michaels_flowering_1999].
The repression of *FLC* during the cold is mediated by a host of different mechanisms that result in epigenetic silencing of the locus.
A long non-coding RNA expressed from the antisense strand at the *FLC* locus is one of the first processes that occur during vernalization[@swiezewski_cold_induced_2009].
Recruitment of Polycomb repressive complex 2 (PRC2) follows.
PRC2 mediates changes to the methylation state of histones, leading to a change in the chromatin structure at the *FLC* locus, repressing its expression[@pien_arabidopsis_2008; @de_lucia_phd_polycomb_2008; @zhao_prevention_2005; @yang_antagonistic_2014].
The recruitment of PRC2 to the *FLC* locus during cold is proposed to involve the product of the *VERNALIZATION INSENSITIVE 3* (*VIN3*) gene[@wood_arabidopsis_2006].
*VIN3* is a plant homoeodomain-finger (PHD) protein upregulated during exposure to cold[@sung_vernalization_2004].
PHD-finger containing proteins mediate histone interactions[@wysocka_phd_2006], and it is thought that the *VIN3* gene product directs the PRC2 complex to the *FLC* locus to induce epigenetic silencing of the gene.
These epigenetic changes are stable across mitotic divisions, allowing the perception of the cold to impact development months after the environmental signal has been perceived[@sheldon_different_2002].
The response of *FLC* at the level of the locus is digital in nature (the locus is either active or repressed), despite showing a quantitative response to cold at the cell population level[@berry_local_2015; @angel_polycomb_based_2011].

A vernalization requirement is a key agronomic trait of *B. napus*, with spring varieties constituting the majority of oilseed rape growth in Canada, Australia, and Northern Europe and winter varieties being grown in Europe and Asia[@friedt_oilseed_2009].
Understanding the requirement for cold is therefore a key part of any analysis of the flowering time control in the crop.
Characterisation of *Brassica* homologues of genes in the vernalization pathway suggest conservation of the pathway in these crops[@li_molecular_2005; @lin_differential_2005; @irwin_functional_2012]
Four copies of *FLC* are present in the *B. rapa* genome[@schranz_characterization_2002], four copies in the *B. oleracea* genome[@okazaki_mapping_2007], and nine copies in *B. napus*[@zou_comparative_2012].
For *FRI*, two copies have been identified in both *B. rapa* and *B. oleracea*[@fadina_molecular_2013; @irwin_functional_2012] and four copies in *B. napus*[@wang_flowering_2011].
Divergence of *Brassica* *FLC* (*BnFLC*) and *FRI* (*BnFRI*) homologues have been revealed in a number of different studies and different *Brassica* crops.
One way in which the genetics of the floral response have been dissected in *Brassica* crops is through the use of association studies.
These studies find correlations between genetic variation and phenotypic variation to try and identify regions of the genome that underlie the phenotypic difference.
Mapping populations are generated by breeding two lines together that exhibit phenotypic differences.
For example, a Doubled Haploid (DH) mapping population generated by crossing Ningyou7, a *B. napus* variety with a slight vernalization response, and Tapidor, the winter variety used in this study, identified genomic regions associated with flowering time that contained *FLC* homologues on chromosomes A10 and A3[@long_flowering_2007; @zou_comparative_2012].
Interestingly, the region on A10 was only associated with unvernalized flowering time as opposed to vernalized flowering time, leading the authors to suggest this locus is one of the determinants of whether a *B. napus* variety is a spring or a winter variety[@zou_comparative_2012].
The *FLC* copy on A2 has also been linked to the vernalization response in *B. napus*[@osborn_comparison_1997].
Using a mapping population derived from two spring varieties of *B. napus* found regions containing *FLC* on chromosomes on A3 and C2 associated with flowering time, suggesting the effect of certain *FLC* copies on flowering time is variety dependent[@raman_molecular_2012; @raman_genetic_2013; @zou_comparative_2012].
Functional divergence of *B. napus* *FLC* homologues has been suggested using transgenic studies.
Different copies of *BnFLC* were found to delay flowering to different extents when expressed in Arabidopsis, indicating conservation in function between the species but divergence in the efficacy of the homologues at repressing the floral transition[@tadege_control_2001].
In *B. rapa*, as in *B. napus*[@osborn_comparison_1997], an *FLC* copy on chromosome A2 emerged as a candidate for flowering time variation[@lou_quantitative_2007; @osborn_comparison_1997; @kole_evidence_2001; @xiao_brassica_2013; @zhao_brflc2_2010].
In addition, Schranz et al. (2002) found that *FLC* copies on A10, A2, and A3 in *B. rapa* influence flowering time[@schranz_characterization_2002].
The C2 copy of *B. oleracea* seems to influence flowering time to a greater extent than the other copies in the species.
A nucleotide difference at the C2 copy of *FLC* in *B. oleracea* reduced the sensitivity of the gene to the environment, resulting in later heading date[@irwin_nucleotide_2016].
Variation in this same homologue was found to account for the majority of flowering time variation in cauliflower[@ridge_role_2015], and was identified as associated with vernalization response in another population[@okazaki_mapping_2007].
Divergence at the protein structure level has been found between *FRI* homologues in *B. oleracea*[@irwin_functional_2012].
That associations to flowering time variation differ between vernalization pathway gene homologues in *Brassica* crops suggests that the copies have diverged, with this being confirmed molecularly in some cases.
However, the roles of copies that do not seem to influence flowering, or influence flowering to a lesser extent, remain elusive.

A potential avenue of subfunctionalization, a partitioning of the roles of an ancestral gene, is spatial subfunctionalization[@conant_turning_2008; @force_preservation_1999].
For example, an ancestral gene may be expressed in both leaves and roots when present as a single copy.
Following a gene duplication event, however, evolution may lead to the presence of leaf specific and root specific gene homologues.
This form of subfunctionalization is an expectation from the duplication-degeneration-complementation model[@force_preservation_1999].
This model for gene evolution posits that after gene duplication, mutations that disrupt cis-regulatory elements that direct tissue specific expression of the gene would be neutral.
This is because the other copy of the gene without the mutation would complement the copy with the mutation.
Over time, mutations in these cis-regulatory elements result in tissue-specific copies of the gene.
This method of subfunctionalization is of particular interest in the context of vernalization, as evidence from a range of sources has found that vernalization acts at both the shoot apex and at the leaves.
Localized cooling experiments in celery (*Apium graveolens*) found that the shoot apex was the site at which vernalization acted in the plant[@curtis_relative_1930].
Similar cooling and grafting experiments also identified the apex as the organ at which vernalization was sensed in *Thlaspi arvense*, a member of the Brassicaceae family like Arabidopsis and the *Brassica* species[@metzger_localization_1988].
However, the authors noted that other tissues, such as the leaves, were still capable of responding to vernalization[@metzger_localization_1988].
In another Brassicaceae family plant, *Lunaria biennis*, plants regenerated from a cutting of vernalized leaves were competent to flower without experiencing cold, indicating that vernalization had occurred in the leaves[@wellensiek_leaf_1961].
Further work indicated that, as opposed to particular tissues, mitotically dividing cells were required for vernalization to take place[@wellensiek_dividing_1962; @wellensiek_dividing_1964].
Results from other species have reinforced that vernalization can be sensed in a range of tissues, such as flower buds[@pierik_lunaria_1966] and roots[@berry_local_2015; @pierik_cichorium_1966], with the general consensus being that the location at which vernalization is sensed in a plant is likely to be species specific[@michaels_memories_2000]
One of the most thorough assessments of the role of *FLC* at both the apex and the leaves was performed by Searle et al. (2006).
By expressing *FLC* in a tissue specific manner, the authors were able to deduce that *FLC* has a dual role in Arabidopsis[@searle_transcription_2006].
Not only does *FLC* induce floral signals in the leaf, through the derepression of *FT* in the leaf,  the product of the gene also acts on floral integrators in the apex, making the regulatory network competent to respond to the signal coming from the leaf[@searle_transcription_2006].
Assessing divergence of *FLC* copies in *B. napus* is of particular interest given the "flowering rheostat" model of *FLC*[@michaels_memories_2000].
This model is based on observations that the delay of flowering mediated by *FLC* is dosage dependent[@lee_late_flowering_1994; @sheldon_flf_1999].
Low or no expression of *FLC* results in a summer annual growth habit, whereas additional copies of *FLC* expressed in Arabidopsis resulted in the plants exhibiting a biennial life strategy[@michaels_memories_2000].
A key question, then, is whether the additional *FLC* copies have been maintained in the *B. napus* genome to maintain gene balance (discussed in chapter \ref{chapter:spring}), or have they diverged to have different expression domains and different effects on the floral transition?

In this chapter, I will discuss the expression of floral integrators and key vernalization pathway genes in the *B. napus* winter variety Tapidor.
The expression of these genes will be compared to the expression of the same genes in Westar in order to address three lines of investigation.
By interpreting global differences between the spring and winter variety, the effect of a requirement for cold on the overall transcriptional landscape is assessed.
This revealed that a requirement for cold has a global effect on the entire transcriptome, delaying expression responses relative to the spring variety.
In addition, the apical transcriptome is determined more by the developmental stage of the plant, whereas the leaf transcriptome seems to be more a consequence of plant age.
The second line of investigation concerns understanding the divergence of vernalization pathway genes.
Specifically, which vernalization pathway genes are candidates for mediating the vernalization response in Tapidor and is there tissue specificity in expression.
Finally, by determining which floral integrator genes are expressed differently in Tapidor relative to Westar, particular copies of floral integrators are assessed for their vernalization sensitivity.
This reveals *FT* and *TFL1* homologues to be most differently expressed, although *B. napus* homologues of other floral integrators also exhibit different patterns of expression in the winter compared to the spring variety.
This provides some evidence that certain floral integrators have diverged to become biased towards, or more sensitive to, particular inputs.
