## Divergence beyond expression {#section:spring:sequence}

The divergence between copies of flowering time genes discussed in this chapter has been in the context of expression differences.
A different expression trace can result from altered cis-regulatory elements, or alternatively, from a change in the transcriptional activators that modulate the expression of the gene.
In previous investigation of how gene function alters after a gene multiplication event, it was proposed that changes to cis-regulatory elements, rather than changes to the amino acid sequence of the protein, may be a common process driving new gene functions.
In addition to changes to the regulation of genes, the molecular activity of the proteins genes encode can also be altered after a gene multiplication event.
In the following section two case studies will be used to illustrate divergence at the sequence level between *Brassica napus* homologues.
The first highlights differences in cis-regulatory elements downstream of the *BrTFL1* genes that correlate with differences observed within the developmental time series and which are supported by mutant studies carried out in Arabidopsis.
The second case study reveals between homologue polymorphisms between the *BrFD* genes.
The identified changes are predicted to influence the dimerization of the protein products, potentially acting as an additional layer of regulatory control that can be leveraged by the plant to coordinate complex flowering responses.
Collectively these two case studies illustrate two ways that changes at the sequence level can lead to divergence of genes following a gene multiplication event.

### *TFL1* cis-regulatory elements {#section:spring:tfl1regulation}

Initial discussions about the methods of gene retention following gene multiplication focused on the aquisition of novel, evolutionarily advantageous, functions to drive gene retention[@ohno_creation_1970].
However, as deleterious mutations occur more frequently than beneficial mutations[@lynch_genetics_1998], this hypothesis is not sufficient to explain gene retention.
A refinement of this thinking was the duplication-degeneration-complementation (DCC) hypothesis[@force_preservation_1999].
The DCC hypothesis posits that multiple copies of genes are maintained through a partitioning of the ancestral gene's functions.
In this hypothesis, different functions (such as expression domains) are determined by the regulatory elements that control expression of the gene.
Deleterious mutations in the regulatory elements of one gene alter the expression of that gene, with the deleterious phenotype of that mutation complemented by the other gene lacking the mutation.
Over evolutionary time regulatory elements continue to accumulate deleterious mutations, with the multiple copies of genes complimenting each other.
In Arabidopsis, 85% of regulatory genes sharing an ancestral gene were found to exhibit expression divergence in agreement with the DCC hypothesis[@duarte_expression_2006].
Using DNase I regulatory footprints to identify regulatory elements upstream of genes, Arsovski et al. (2015) found that the vast majority of paralogous gene pairs were either partly or fully divergence in the regulatory elements they possessed[@arsovski_evolution_2015].
Cis-regulatory element changes are therefore thought to be one of the most important driving factors for the evolution of novel gene functions.

Although DNase I footprinting is able to identify putative regulatory elements, determining the effects those regulatory elements have on gene expression requires more in depth investigation.
Such an in depth analysis has recently been conducted in Arabidopsis for the *TFL1* gene[@serrano_mislata_separate_2016].
Sequence conservation between the Arabidopsis *TFL1* gene and other *TFL1*-like genes from other species was used to identify conserved regions upstream and downstream of the gene.
Through mutational analysis and transgenic experiments Serrano-Mislata et al. were able to determine that the regions of conservation corresponded to cis-regulatory elements.
Moreover, the regulatory effects of each cis-regulatory element was uncovered, such that the spatiotemporal effects of removing elements was determined.
These results from Arabidopsis, and the *unique* pattern of regulatory module assignment observed for the *BrTFL1* genes in *Brassica napus* (Figure \ref{figure:2xx:soc1leaf}), prompted me to investigate the cis-regulatory elements downstream of the *BrTFL1* genes.

#### Cis-regulatory element variation downstream of *BrTFL1* genes {#section:spring:tfl1conservation}

![**Title.** Description.](figuredirectory/sequence_conservation_qpcr.pdf){#fig:spring:tfl1conservation}

Sequence conservation between the *BrTFL1* genes and *AtTFL1* was calculated in the 5' and 3' intergenic regions surrounding the genes and within the gene itself.
Several conserved regions within the intergenic regions were identified (Figure \ref{fig:spring:tfl1conservation}).
Serrano-Mislata et al. (2016) identified seven regions of interspecies sequence conservation surrounding the *AtTFL1* gene (denoted by green letters in figure \ref{fig:spring:tfl1conservation}) and five regions that were experimentally verified to be cis-regulatory elements (denoted by blue numerals in figure \ref{fig:spring:tfl1conservation}).
Focussing the analysis on the five experimentally verified cis-regulatory elements, I find differences in the extent of sequence conservation within these regions between the *BrTFL1* genes.
The high sequence conservation in region II and IV of *BrTFL1.C3* and *BrTFL1.A10* suggests these two copies of the gene possess Arabidopsis-like cis-regulatory elements.
Conversely, the lack of sequence conservation in these two regions in the *BrTFL1.C2* and *BrTFL1.Cnn.Random* copies suggests these copies are lacking such regulatory sequence.
Maximal sequence conservation within region III is below 50% in the *BrTFL1.Cnn.Random* copy, while this value is above 70% for the other three copies (81%, 87%, and 78% for *BnaTFL1.A10*, *BnaTFL1.C2*, and *BnaTFL1.C3* respectively.
Interestingly, the area of significant sequence conservation in *BnaTFL1.C2* (154 bases) and *BnaTFL1.A10* (162 bases) is decreased compared to that of *BnaTFL1.C3* (273 bases) copies, potentially suggesting the cis-regulatory elements in the former are incomplete.
Considering regions identified as conserved across species by Serrano-Mislata et al. (2016), but not experimentally implicated in the regulatory control of *AtTFL1* (green shading in Figure \ref{fig:spring:tfl1conservation}), we observe sequence divergence in region G.
*BnaTFL1.A10* exhibits high sequence conservation relative to Arabidopsis across this entire region, while *BnaTFL1.C3* shows conservation over ~50% of the region.
As with regions II and IV, *BnaTFL1.C2* and *BnaTFL1.Cnn.Random* lack conserved sequence in region G. 
We also identify a region of conservation not annotated in the previous analysis of *AtTFL1* cis-regulatory elements. This region, situated ~600 bp upstream of the transcription start site of *AtTFL1*, shows ~80% sequence conservation relative to Arabidopsis in *BnaTFL1.A10*, *BnaTFL1.C2* and *BnaTFL1.Cnn.Random*.
In *BnaTFL1.C3*, sequence conservation in this newly identified region is ~55%.

#### Variation in cis-regulatory elements correlates with expression divergence {#section:spring:tfl1expdivergence}

The experiments conducted to identify the regulatory effects of the cis-regulatory elements downstream of *AtTFL1* consisted of transgenic and mutational studies[@serrano_mislata_separate_2016].
Insertion lines that disrupted cis-regulatory elements and transgenic lines transformed with reporter genes whose expression was driven by different combinations of the regulatory elements were used to dissect the role each element played in directing the correct spatiotemporal expression domain of *AtTFL1*.
As certain *BnaTFL1* genes lacked the identified downstream regulatory elements, I compared the expression of *AtTFL1* in the mutant and transgenic lines of Serrano-Mislata et al. to the expression of the *BnaTFL1* genes during the developmental transition.
The *BnaTFL1* genes that increase in expression during the floral transition (*BnaTFL1.C3* and *BnaTFL1.A10*) both show high sequence conservation in region II.
Conversely, *BnaTFL1.C2* and *BnaTFL1.Cnn.Random* both lack sequence conservation in region II and are not unregulated during the floral transition.
Region II was found to be necessary for the upregulation of *AtTFL1* during the floral transition in Arabidopsis[@serrano_mislata_separate_2016], which correlates with the expression traces of *BnaTFL1* genes during the developmental time series.
Another region showing a similar presence-absence pattern between the *BnaTFL1* genes as region II is region IV.
In Arabidopsis, this region corresponds to a cis-regulatory element responsible for driving the expression of *AtTLF1* in the inflorescence meristem.
Potentially the presence or absence of this region also contributes to the expression differences observed between the *BnaTFL1* genes.
Region III was found to be important for the expression of *AtTFL1* in the lateral meristems of the plant[@serrano_mislata_separate_2016].
Sequence conservation within region III is below 50% for the *BnaTFL1.Cnn.Random* gene.
This finding predicts that this particular copy, therefore, would not be expressed in the lateral meristems in *Brassica napus*.

#### Quantitative PCR validation of *BnaTFL1* RNA-Seq expression levels {#section:spring:tfl1qpcr}

The above observations of gene expression correlating with the presence and absence of cis-regulatory elements is dependent on the accuracy of the RNA-Seq results.
Although findings presented in section \ref{section:spring:alignreadexplevel} suggest that spurious expression levels as a result of read mismapping is a rare occurrence (Figure \ref{figure:206:uniquefpkm}), I wished to measure expression of the *BnaTFL1* genes in a copy specific manner.
To do this, quantitative PCR (qPCR) primers were designed in a copy specific manner for the four copies of *BnaTFL1*.
The qPCR results obtained show strong similarity to the expression traces derived from the RNA-Seq data (Figure \ref{fig:spring:tfl1conservation}).
As the qPCR primers designed were copy specific, I can therefore be confident that the expression trace divergence between the *BnaTFL1* copies observed in the RNA-Seq data is not an artefact of read mismapping or incomplete gene models.

### *FD* dimerization {#section:spring:fdprotein}

Dimerization as a mechanism of control within protein mediated signalling pathways has been discussed and reported for many years.
In the case of DNA binding proteins, dimerization may have a number of effects.
Monomers can have different DNA binding preferences, such that a heterodimer of two different monomers would bind to a different DNA recognition sequence than either homodimer.
The production of non-functional transcription factor monomers, such as monomers without DNA-binding domains, may sequester functional monomers and alter the expression of the targets of the transcription factor.
Finally, monomers may alter the composition of the protein complex recruited to the DNA through variation in protein-protein interaction domains.
Transcription factor dimerization as a method of modulating gene expression has been reported previously.
In mouse, it was found that the helix-loop-helix (HLH) protein Id formed protein-protein interactions with three other HLH proteins (MyoD, E12, and E47) and that the heterodimers involving Id were compromised in their ability to bind to the DNA recognition sequences[@benezra_protein_1990].
In flower development, the ABCE model proposes that the composition of the protein tetramers directs the formation of different floral structures[@krizek_molecular_2005].
Finally, *BROTHER OF FT AND TFL1* (*BFT*) produces a protein that competes with FT for binding to FD, and this cometition mediates the delay in flowering that salt stress induces[@ryu_arabidopsis_2014].
These examples illustrate that regulating the composition of a protein complex that in turn influences gene expression for strict control of transcriptional programs.

The function of the *FD* gene in Arabidopsis has been discussed previously in section \ref{section:spring:fd}.
The FD protein is a bZIP transcription factor that interacts with both FT TFL1 proteins to mediate their association with DNA[@abe_fd_2005; @jaeger_interlocking_2013].
FD protein contains a basic region leucine zipper (bZIP) domain, making it a member of the bZIP transcription factors.
This family of transcription factors form dimers that are mediated by a coiled-coil structure of two $\alpha$-helicies.
The interaction domain between the two helicies is hydrophobic as a result of the presence of the amino acid leucine, which gives the dimerization domain its name.
In addition to the hydrophobic core of the interaction interface, charged amino acid residues adjacent to the core influence the dimerization affinity.
In the case of Fos and Jun, two proteins from humans, the dimerization of the two proteins is essential to their function.
Jun is able to form functional homodimers, whereas Fos requires the formation of heterodimers with Jun in order to bind to DNA.
In depth protein interaction studies determined that the interaction of Fos and Jun was controlled through the interactions between the charged residues adjacent to the the hydrophobic core of the interaction domain; with Fos homodimers being considerably less favourable than the Jun homodimer or the heterodimer of the two proteins[@john_two_1994].
Transcription factors that have bZIP domains bind DNA via a basic region that consists of an $\alpha$-helix that interacts with the major groove of DNA.
Of the six copies of *BnaFD* that are expressed in the apex in *Brassica napus*, five of them share similar expression traces (Figure \ref{figure:2xx:fdapex}).
As these genes are coregulated, it is likely that their protein products are present in the cell at the same time, and would have the potential to interact to form dimers.
To determine whether dimerization of different FD proteins could potentially be used as a regulatory mechanism, dimerization affinities between the six copies of *BnaFD* were predicted.
Differential binding affinities were detected, with certain dimers being more favourable than others.
The regulatory ramifications of this represent a mechanism by which the multiple copies of *BnaFD* have been maintained in the *Brassica napus* genome.
It also highlights that although genes may exhibit similar expression patterns, that does not indicate they are acting redundantly in the plant.

#### Protein sequence divergence exists between the six *BnaFD* copies {#section:spring:bnafddivergence}

![**Multiple sequence alignment of the *Arabidopsis thaliana* and *Brassica napus* FD proteins** The indicated regions of the protein are defined as in REF. The heptad structure of the leucine zipper region is displayed below the alignment with the coil interior amino acids and amino acids which are capable of forming electrostatic interactions shaded grey and black respectively.](figuredirectory/brassica_sequence_alignment.pdf){#figure:spring:brassicasequence}

In order to assess the extent of amino acid divergence between six copies of *BnaFD*, their predicted protein sequences were determined and aligned (Figure \ref{figure:spring:brassicasequence}).
To identify polymorphisms likely to affect the molecular function of the protein, the results of a comparative study of FD-like genes from many species were used[@tsuji_functional_2013].
The Arabidopsis FD protein was found to have four conserved regions: the A region, the LSL region, the bZIP region (composed of the basic region and a leucine zipper region) and the SAP region.
Looking at the same regions in *Brassica napus* (Figure \ref{figure:spring:brassicasequence}) identifies a number of amino acid changes and deletions in the A region, with four different forms of the region present in the six BnaFD proteins.
Comparing the BnaFD proteins to the AtFD protein reveals that, in the A region, BnaFD.A8 and BnaFD.C3 show the greatest amino acid sequence similarity to the AtFD protein, with only a single amino acid change present.

The LSL region displays no amino acid variation within the *Brassica napus* FD proteins or between species.
This is consistent with the findings of Tsuji et al. (2013), who suggested that the LSL region was indicative of FD-like proteins that played a role in the floral pathway[@tsuji_functional_2013].

In the SAP region, there are again a number of amino acid changes between the different copies of the *BnaFD* genes (Figure \ref{figure:spring:brassicasequence}).
Of note is the amino acid polymorphism at position 287 between a theonine and serine.
This position in *Arabidopsis thaliana* becomes phosphorylated and is important for the binding of FD to the protein FT in Arabidopsis as mutation of the threonine to an alanine disrupts complex formation[@abe_fd_2005].
Changing the threonine to a serine was found to not affect FD binding to FT in Arabidopsis, although potentially different kinases are responsible for the phosphorylation of the different residues.

Finally, there are a number of amino acid changes within the bZIP region, in both the DNA binding basic region and the leucine zipper region that is the dimerization interface between FD monomers.
To confirm that the polymorphisms observed in this important protein domain are present in the spring variety used for the developmental time series, copy specific primers were designed to amplify the bZIP domain of each of the six *BnFD* genes.
Sanger sequencing was able to confirm that the polymorphisms found in the Darmor-*bzh* reference genome are also present in Westar, the spring variety that was used to generate the developmental transcriptome.
As the effects these amino acid changes have on the function of bZIP transcription factors is better understood, these changes will be discussed in more detail in later sections.

There are many amino acid changes identified both within the copies of *BnFD* and between the copies and *AtFD*.
A number of polymorphisms occur within the regions identified as being conserved within FD-like proteins, and these are good candidates for potentially affecting the function of the proteins.
While there are other polymorphisms outside of these conserved regions, there is currently no reported evidence to suggest that any of these other polymorphisms are having a functional effect on the proteins.
Indeed, besides the bZIP transcription factor, the roles the identified conserved regions play in the function of FD are also yet to be fully determined.

#### Polymorphisms in the DNA binding interface have the potential to affect binding affinities {#section:spring:fddnabinding}

![**Protein structure of the *Brassica napus* FD genes complexed with DNA** The protein structure of the CREB protein from REF was mutated to match the amino acids which are present in the *Brassica napus* FD proteins. The green dashed lines indicate hydrogen bonding between atoms. The colour scheme for atoms is as follows: white (carbon), dark blue (nitrogen), yellow (phosphorus), red (oxygen) and light blue (hydrogen).](figuredirectory/protein_structure.pdf){#figure:spring:proteinstructure}

The basic region of bZIP transcription factors forms hydrogen bonds within the major groove of DNA and consists of the protein-DNA interaction interface.
Within this region of the BnFD proteins, there are two positions that exhibit between copy polymorphisms; positions 222 and 223 (Figure \ref{figure:spring:brassicasequence}).
To investigate the potential effects of these mutations on the DNA binding properties of BnFD, an available crystal structure of a bZIP transcription factor bound to DNA was used.
The crystal structure of the mammalian cAMP responsive element-binding protein (CREB) bZIP transcription factor bound to DNA revealed that position 222 is important as the amino acid side chain forms a hydrogen bond with the DNA backbone (@schumacher_structure_2000).
Mapping the amino acids in the basic region from the BnFD proteins onto the crystal structure of the CREB transcription factor revealed that changing the amino acid in position 222 from an arginine to a histidine disrupts hydrogen bond formation between the protein and the DNA (Figure \ref{figure:spring:proteinstructure}).
Having a histidine or an asparagine in the 223 position does not seem to affect the hydrogen bonding in the $\alpha$-helix or between the protein and DNA.
Therefore, the amino acid polymorphisms present in the basic region of BnFD proteins potentially affects the DNA binding affinity of the monomers.

#### Amino acid differences in the leucine zipper region of BnaFD proteins is predicted to alter dimerization affinity {#section:spring:fddimerizationprediction}

![**Helical wheel representation of the homodimers and heterodimer possible with the *Brassica napus* genes XLOC_037807 and XLOC_061874** The blue coloured amino acids have positively charged side chains, the red coloured animo acids have negatively charged side chains and the orange amino acids have polar side chains. The blue and red dotted lines between helical wheels indicate attractive and repulsive electrostatic charges between the two helicies respectively.](figuredirectory/helical_wheels.pdf){#figure:spring:helicalwheels}

Considerable amino acid differences between the BnFD proteins occur in the leucine zipper region.
To determine whether these differences have the potential to alter the dimerization affinity of the proteins, the amino acid polymorphisms were assessed in the context of the coiled-coil dimerization interface (Figure \ref{figure:spring:helicalwheels}).
Previous studies of bZIP transcription factors have revealed that amino acid residues in the `e` and `g` positions of the heptad of the $\alpha$-helix are important in the determination of dimerization specificity[@ellenberger_getting_1994; @john_two_1994].
Specifically, when the proteins form a coiled-coil structure, the side chain of an amino acid in the `e` position on one $\alpha$-helix is able to form electrostatic bonds with the side chain of an amino acid in the `g` position on the other $\alpha$-helix.
This is illustrated in the helical wheel representations in Figure \ref{figure:spring:helicalwheels} that represent the positions of amino acids in the coiled-coil.
An example of this, using the positions in Figure \ref{figure:spring:proteinstructure} as a reference, is residue 250 (in the `g` position of the heptad) has the capacity to form electrostatic interactions with residue 255 (in the `e` position of the heptad) when in the coiled-coil arrangement.
Therefore, the charge these residues carry is a factor which determines the dimerization energy between bZIP proteins.
Positions 250, 255, 262 and 271 are all in either the `e` or `g` positions of the heptads and show amino acid polymorphisms which alter the charge of the amino acid side chains.
The effect this has on the predicted electrostatic interactions is illustrated in Figure \ref{figure:spring:helicalwheels}.
The BnFD.C1 homodimer and the BnFD.C1-BnFD.C7 heterodimer are both predicted to have attractive electrostatic interactions between the two monomers, while a repulsive force is predicted for the BnFD.C7 homodimer (Figure \ref{figure:spring:helicalwheels}).
These polymorphisms suggest that certain dimer combinations of the BnaFD proteins will be more favoured than others.

![**Heatmap of the dimerization affinity scores computed between *Arabidopsis thaliana* and *Brassica napus* FD leucine zipper regions** The machine learning algorithm discussed in the main text was used to score the dimerization affinity of the leucine zipper regions of the proteins. Higher scoring dimers are dimers more likely to have dimerization energies which are favourable. The leucine zipper regions used for the analysis correspond to the region indicated in Figure @fig:spring:brassicasequence](figuredirectory/fd_dimer_heatmap.pdf){#figure:spring:bnafdheatmap}

The sequence analysis suggests that the amino acid polymorphisms observed in the `e` and `g` positions of the heptad may affect the dimerization affinity of the proteins.
To investigate this in a more quantitative manner, a previously published machine learning algorithm[@potapov_data_driven_2015] was used to score the potential interaction affinity of pairs of proteins (Figure @fig:spring:heatmapwofusjun).
The interaction scores range from a minimum of 4.3 to a maximum of 7.2, with the higher interaction scores indicating a higher predicted affinity.
To put these scores into context, the dimerization of the bZIP transcription factors Fos and Jun have been extensively studied in terms of their dimerization affinity[@john_two_1994].
In particular, the Fos-Jun heterodimer is more thermally stable than either the Fos or the Jun homodimer, with the Fos homodimer being particularly unfavourable.
Using the machine learning scoring algorithm of Keating et al. (2015), Fos homodimers score 6.2, Jun homodimers score 6.3 and Fos-Jun heterodimers score 8.8.
The score range for Fos and Jun dimers is 2.6, a similar range as that observed for the BnaFD proteins.
Therefore, the differences in interactions scores observed between the BnFD proteins are large enough to suggest a functional effect.
The heatmap of these interaction scores (Figure \ref{figure:spring:bnafdheatmap}) illustrates that the six BnaFD genes fall into three interaction groups.
BnFD.C1 and BnFD.A1 form a group which have a higher affinity for forming dimers between themselves than with the remaining four proteins.
BnFD.A8 and BnFD.C3.Random have a mild affinity to form dimers between themselves, but have a higher affinity to form dimers with both BnFD.C1 and BnFD.A1.
Finally, BnFD.Ann.Random and BnFD.C7 have the lowest affinity to form dimers between themselves relative to the other dimers tested, and have the highest affinity to form dimers with both BnFD.C1 and BnFD.A1.
The machine learning approach predicts that the six copies of BnFD have variation in their dimerization affinities, with four of the six copies predicted to form more stable heterodimers than homodimers.
The range of interaction scores predicted for the BnFD proteins is similar in size to the range of interaction scores predicted for the Fos and Jun proteins, suggesting that the predicted differences have the potential to be biologically relevant.

#### bZIP transcription factors in other organisms reveal that charge influencing amino acid polymorphisms in the `e` and `g` heptad positions are observed in other species

![**Multiple sequence alignment of the *Arabidopsis thaliana* FD protein and its orthologues in *Brassica napus*, *Glycine max*, *Musa acuminata subsp. malaccensis* and *Medicago truncatula*** The alignment is limited to the same regions identified by REF and displayed in Figure @fig:spring:brassicasequence.](figuredirectory/ensembl.pdf){#figure:spring:ensembl}

![**Heatmap of the dimerization affinity scores computed between *Musa acuminata* FD orthologue leucine zipper regions**](figuredirectory/fd_dimer_heatmap_musa_acuminata.pdf){#fig:spring:heatmapmusaacuminata}

![**Multiple sequence alignment of the *Arabidopsis thaliana* FD protein, its orthologues in *Brassica napus* and proteins with high sequence similarity from the *Zea mays* and *Triticum aestivum* proteomes** The alignment is limited to the same regions identified by REF and displayed in Figure @fig:spring:brassicasequence. The *Zea mays* proteins plotted were chosen by selecting the *Zea mays* protein with the highest sequence similarity to the *Arabidopsis thaliana* FD protein and then including the paralogues of the gene as identified in the Ensembl Plants database. The *Triticum aestivum* proteins were identified in the same way, except that in addition to the paralogues, the homeologues of the proteins were also included.](figuredirectory/brassica_wheat_maize_fd.pdf){#fig:spring:wheatmaizefd}

![**Heatmap of the dimerization affinity scores computed for the leucine zipper region between the *Zea mays* protein with the highest protein sequence similarity to the *Arabidopsis thaliana* FD protein and its paralogues**](figuredirectory/fd_dimer_heatmap_zea_mays.pdf){#fig:spring:heatmapzeamays}

![**Heatmap of the dimerization affinity scores computed for the leucine zipper region between the *Triticum aestivum* protein with the highest protein sequence similarity to the *Arabidopsis thaliana* FD protein, its paralogues and the homeologues for all proteins**](figuredirectory/fd_dimer_heatmap_triticum_aestivum.pdf){#fig:spring:heatmaptriticumaestivum}

To investigate whether polymorphisms influencing dimerization affinity were a common occurrence in organisms where gene multiplication events have occurred, sequences of *FD* orthologues identified in the Ensembl Plants database were aligned.
Only those species containing multiple *FD* orthologues in the genome are displayed in Figure \ref{figure:spring:ensembl}.
Looking at the leucine-zipper region identifies similar charge influencing polymorphisms in the `e` and `g` heptad positions between the genes within a species.
Charge influencing polymorphisms in the `e` and `g` heptad positions are present in *Glycine max* at positions 360, 362 and 381, *Musa acuminata* at positions 362, 367, 374 and 376 and *Medicago truncatula* at positions 360, 362, 367 and 381.
Likewise, *Zea mays* and *Triticum aestivum* proteins with high sequence similarity to Arabidopsis *FD* also exhibit polymorphisms in the `e` and `g` heptad positions that alter the charge of the amino acid side chain.
The machine learning algorithm[@potapov_data_driven_2015] predicts considerable variation in the dimerization affinity for the identified FD-like proteins, with the range of scores being similar to the range identified for the BnFD proteins.
These findings suggest that changes to the dimerization affinity of bZIP transcription factors play a considerable role in their evolution following gene multiplication in many a number of species.

#### Computational modelling reveals that the modulation of dimerization affinity for dimerizing transcription factors...

![**test**](figuredirectory/fd_dimerization.pdf){#figure:spring:dimerizationmodelling}

\ch{a + a <=>[7][1] aa}

\ch{a + b <=>[k][1] ab}

\ch{b + b <=>[0.5/4/7][1] bb}

To test some of the regulatory repercussions of altered dimerization, a system of ordinary differential equations were used to model a system of dimerization reactions.
Two different monomer types were modelled, with the monomers able to form homodimers and a heterodimer.
To test how the behaviour of the system depends on the dimerization affinities, three different reaction rates for the homodimerization of the `b` species were tested; 0.5, 4.0 and 7.0.
For each of these rates, the heterodimerization rate for the monomers was varied and the steady state concentrations of the various species calculated (Figure \ref{figure:spring:dimerizationmodelling}).
For a `b` homodimerization rate of 7.0, the two homodimer species have the same steady state concentrations at all heterodimerization rates, as expected given that all dimerization reactions have the same reaction rates.
When all dimerization rates are 7.0, the steady state concentrations of all dimers are identical.
By setting the `b` homodimerization rate to 0.5, the `bb` homodimer is disfavoured, with an observed increase in the steady state concentration of the undimerizaed `b` monomer.
This has has an interesting interaction with the steady state concentration of the heterodimer.
Above a heterodimer formation rate of ~2.0, the heterodimer becomes more favourable than either of the homodimers.
This implies that the presence of the `b` monomer, as a result of an unfavourable `bb` homodimer, favours the formation of the `ab` heterodimer rather than the `aa` homodimer, despite the forward reaction rate of the `aa` homodimer being 2.5 times greater than the `ab` heterodimer formation rate.
A similar pattern is observed when less extreme `bb` homodimer formation rate of 4.0 is used in the modelling.

#### Conclusions

The two examples of sequence divergence between copies of flowering time genes in *Brassica napus* presented here demonstrate the utility of the developmental transcriptome.
In the case of the *BnaTFL1* genes, the expression traces of the genes and the work from Arabidopsis allow the different patterns of sequence conservation downstream of the *Brassica napus* genes to be considered in context.
The conservation suggests that the spatiotemporal domains of expression defined by the cis-regulatory elements is conserved between *Brassica napus* and Arabidopsis.
Moreover, the case study potentially provides an example of cis-regulatory element changes driving the development of novel gene functions.
Although the relationship between the sequence conservation downstream of the *BnaTFL1* genes and the expression traces exhibited by the genes is correlative, it provides a hypothesis to be tested by future studies.
Transgenic investigations of Arabidopsis could be used to test such hypotheses, such as transforming *tfl1* null mutant Arabidopsis lines with the *BnaTFL1* genes.
If these insertions also included the downstream intergenic regions, the functional conservation of the cis-regulatory elements could be established.

For the *BnaFD* genes, the expression traces suggest that the genes are expressed in similar expression domains.
As FD is a bZIP containing protein, this prompted me to investigate whether the different copies of the gene could potentially dimerize.
Computational predictions suggest that this is the case, with the affinity of dimers differing depending on the monomers that form the dimer.
Looking at *FD* homologues in other species reveals that variation in dimerization affinity might be a common direction of divergence for bZIP transcription factors that are present as multiple copies in the genome.
The regulatory ramifications of this are that a change in the concentration of one monomer has the potential of shifting the concentrations of all possible dimers.
If the dimers regulate genes in different ways, then the change in concentration of one monomer would result in large changes in transcription.
A caveat to this analysis would be that the spatial expression domains in which the *BnaFD* proteins are expressed may be too small to be resolved by the sampling method used.
Therefore, although five of the six *BnaFD* genes are assigned to the same regulatory module (Figure \ref{figure:spring:fdapex}), they may not be expressed in the same cells.
If this is the case, then the dimerization dynamics and the potential regulatory consequences of them would not be applicable.
A method of determining if the different *BnaFD* proteins do interact *in vivo* might be to introduce a transgene with the same regulatory sequences with a protein tag attached.
Using enrichment techniques and proteomics the *in vivo* interaction partners of the tagged protein could be determined.
The other caveat with this finding is that although certain dimers may be more favourable, the different dimers may not have differing functions.
If this was the case, then the hypothesised regulatory mechanism outlined would not be applicable.
This could be tested by expressing a transgenic *FD* gene in Arabidopsis, where the two FD protein monomers are forced to dimerize through a linker.
