## Divergence beyond expression {#section:spring:sequence}

The divergence between copies of flowering time genes discussed in this chapter has been in the context of expression differences.
A different expression trace can result from altered cis-regulatory elements, or alternatively, from a change in the transcriptional activators that modulate the expression of the gene.
In previous investigation of how gene function alters after a gene multiplication event, it was proposed that changes to cis-regulatory elements, rather than changes to the amino acid sequence of the protein, may be a common process driving new gene functions.
In addition to changes to the regulation of genes, the molecular activity of the proteins genes encode can also be altered after a gene multiplication event.
In the following section two case studies will be used to illustrate divergence at the sequence level between *Brassica napus* homologues.
The first highlights differences in cis-regulatory elements downstream of the *BrTFL1* genes that correlate with differences observed within the developmental time series and which are supported by mutant studies carried out in Arabidopsis.
The second case study reveals between homologue polymorphisms between the *BrFD* genes.
The identified changes are predicted to influence the dimerization of the protein products, potentially acting as an additional layer of regulatory control that can be leveraged by the plant to coordinate complex flowering responses.
Collectively these two case studies illustrate two ways that changes at the sequence level can lead to divergence of genes following a gene multiplication event.

### *TFL1* cis-regulatory elements

Initial discussions about the methods of gene retention following gene multiplication focused on the aquisition of novel, evolutionarily advantageous, functions to drive gene retention[@ohno_creation_1970].
However, as deleterious mutations occur more frequently than beneficial mutations[@lynch_genetics_1998], this hypothesis is not sufficient to explain gene retention.
A refinement of this thinking was the duplication-degeneration-complementation (DCC) hypothesis[@force_preservation_1999].
The DCC hypothesis posits that multiple copies of genes are maintained through a partitioning of the ancestral gene's functions.
In this hypothesis, different functions (such as expression domains) are determined by the regulatory elements that control expression of the gene.
Deleterious mutations in the regulatory elements of one gene alter the expression of that gene, with the deleterious phenotype of that mutation complemented by the other gene lacking the mutation.
Over evolutionary time regulatory elements continue to accumulate deleterious mutations, with the multiple copies of genes complimenting each other.
In Arabidopsis, 85% of regulatory genes sharing an ancestral gene were found to exhibit expression divergence in agreement with the DCC hypothesis[@duarte_expression_2006].
Using DNase I regulatory footprints to identify regulatory elements upstream of genes, Arsovski et al. (2015) found that the vast majority of paralogous gene pairs were either partly or fully divergence in the regulatory elements they possessed[@arsovski_evolution_2015].
Cis-regulatory element changes are therefore thought to be one of the most important driving factors for the evolution of novel gene functions.

Although DNase I footprinting is able to identify putative regulatory elements, determining the effects those regulatory elements have on gene expression requires more in depth investigation.
Such an in depth analysis has recently been conducted in Arabidopsis for the *TFL1* gene[@serrano_mislata_separate_2016].
Sequence conservation between the Arabidopsis *TFL1* gene and other *TFL1*-like genes from other species was used to identify conserved regions upstream and downstream of the gene.
Through mutational analysis and transgenic experiments Serrano-Mislata et al. were able to determine that the regions of conservation corresponded to cis-regulatory elements.
Moreover, the regulatory effects of each cis-regulatory element was uncovered, such that the spatiotemporal effects of removing elements was determined.
These results from Arabidopsis, and the *unique* pattern of regulatory module assignment observed for the *BrTFL1* genes in *Brassica napus* (Figure \ref{figure:2xx:soc1leaf}), prompted me to investigate the cis-regulatory elements downstream of the *BrTFL1* genes.

#### Cis-regulatory element variation downstream of *BrTFL1* genes {#section:spring:tfl1conservation}

![**Title.** Description.](figuredirectory/sequence_conservation_qpcr.pdf){#fig:spring:tfl1conservation}

Sequence conservation between the *BrTFL1* genes and *AtTFL1* was calculated in the 5' and 3' intergenic regions surrounding the genes and within the gene itself.
Several conserved regions within the intergenic regions were identified (Figure \ref{fig:spring:tfl1conservation}).
Serrano-Mislata et al. (2016) identified seven regions of interspecies sequence conservation surrounding the *AtTFL1* gene (denoted by green letters in figure \ref{fig:spring:tfl1conservation}) and five regions that were experimentally verified to be cis-regulatory elements (denoted by blue numerals in figure \ref{fig:spring:tfl1conservation}).
Focussing the analysis on the five experimentally verified cis-regulatory elements, I find differences in the extent of sequence conservation within these regions between the *BrTFL1* genes.
The high sequence conservation in region II and IV of *BrTFL1.C3* and *BrTFL1.A10* suggests these two copies of the gene possess Arabidopsis-like cis-regulatory elements.
Conversely, the lack of sequence conservation in these two regions in the *BrTFL1.C2* and *BrTFL1.Cnn.Random* copies suggests these copies are lacking such regulatory sequence.
Maximal sequence conservation within region III is below 50% in the *BrTFL1.Cnn.Random* copy, while this value is above 70% for the other three copies (81%, 87%, and 78% for *BnaTFL1.A10*, *BnaTFL1.C2*, and *BnaTFL1.C3* respectively.
Interestingly, the area of significant sequence conservation in *BnaTFL1.C2* (154 bases) and *BnaTFL1.A10* (162 bases) is decreased compared to that of *BnaTFL1.C3* (273 bases) copies, potentially suggesting the cis-regulatory elements in the former are incomplete.
Considering regions identified as conserved across species by Serrano-Mislata et al. (2016), but not experimentally implicated in the regulatory control of *AtTFL1* (green shading in Figure \ref{fig:spring:tfl1conservation}), we observe sequence divergence in region G.
*BnaTFL1.A10* exhibits high sequence conservation relative to Arabidopsis across this entire region, while *BnaTFL1.C3* shows conservation over ~50% of the region.
As with regions II and IV, *BnaTFL1.C2* and *BnaTFL1.Cnn.Random* lack conserved sequence in region G. 
We also identify a region of conservation not annotated in the previous analysis of *AtTFL1* cis-regulatory elements. This region, situated ~600 bp upstream of the transcription start site of *AtTFL1*, shows ~80% sequence conservation relative to Arabidopsis in *BnaTFL1.A10*, *BnaTFL1.C2* and *BnaTFL1.Cnn.Random*.
In *BnaTFL1.C3*, sequence conservation in this newly identified region is ~55%.

#### Variation in cis-regulatory elements correlates with expression divergence {#section:spring:tfl1expdivergence}

The experiments conducted to identify the regulatory effects of the cis-regulatory elements downstream of *AtTFL1* consisted of transgenic and mutational studies[@serrano_mislata_separate_2016].
Insertion lines that disrupted cis-regulatory elements and transgenic lines transformed with reporter genes whose expression was driven by different combinations of the regulatory elements were used to dissect the role each element played in directing the correct spatiotemporal expression domain of *AtTFL1*.
As certain *BnaTFL1* genes lacked the identified downstream regulatory elements, I compared the expression of *AtTFL1* in the mutant and transgenic lines of Serrano-Mislata et al. to the expression of the *BnaTFL1* genes during the developmental transition.
The *BnaTFL1* genes that increase in expression during the floral transition (*BnaTFL1.C3* and *BnaTFL1.A10*) both show high sequence conservation in region II.
Conversely, *BnaTFL1.C2* and *BnaTFL1.Cnn.Random* both lack sequence conservation in region II and are not unregulated during the floral transition.
Region II was found to be necessary for the upregulation of *AtTFL1* during the floral transition in Arabidopsis[@serrano_mislata_separate_2016], which correlates with the expression traces of *BnaTFL1* genes during the developmental time series.
Another region showing a similar presence-absence pattern between the *BnaTFL1* genes as region II is region IV.
In Arabidopsis, this region corresponds to a cis-regulatory element responsible for driving the expression of *AtTLF1* in the inflorescence meristem.
Potentially the presence or absence of this region also contributes to the expression differences observed between the *BnaTFL1* genes.
Region III was found to be important for the expression of *AtTFL1* in the lateral meristems of the plant[@serrano_mislata_separate_2016].
Sequence conservation within region III is below 50% for the *BnaTFL1.Cnn.Random* gene.
This finding predicts that this particular copy, therefore, would not be expressed in the lateral meristems in *Brassica napus*.

#### Quantitative PCR validation of *BnaTFL1* RNA-Seq expression levels {#section:spring:tfl1qpcr}

The above observations of gene expression correlating with the presence and absence of cis-regulatory elements is dependent on the accuracy of the RNA-Seq results.
Although findings presented in section \ref{section:spring:alignreadexplevel} suggest that spurious expression levels as a result of read mismapping is a rare occurrence (Figure \ref{figure:206:uniquefpkm}), I wished to measure expression of the *BnaTFL1* genes in a copy specific manner.
To do this, quantitative PCR (qPCR) primers were designed in a copy specific manner for the four copies of *BnaTFL1*.
The qPCR results obtained show strong similarity to the expression traces derived from the RNA-Seq data (Figure \ref{fig:spring:tfl1conservation}).
As the qPCR primers designed were copy specific, I can therefore be confident that the expression trace divergence between the *BnaTFL1* copies observed in the RNA-Seq data is not an artefact of read mismapping or incomplete gene models.

### *FD* dimerization

Dimerization as a mechanism of control within protein mediated signalling pathways has been discussed and reported for many years.
In the case of DNA binding proteins, dimerization may have a number of effects.
Monomers may have different DNA binding preferences, such that a heterodimer of two different monomers would tend to bind to a different DNA recognition sequence than either homodimer.
The production of non-functional transcription factor monomers, such as monomers without DNA-binding domains, may sequester functional monomers and alter the expression of the targets of the transcription factor.
Monomers may have other protein-protein interaction interfaces other than the dimerization interface.
If monomers were to have different protein-protein interaction interfaces, then different dimers may alter the composition of the protein complex recruited to the DNA.
Transcription factor dimerization as a method of modulating gene expression has been reported previously.
In mouse, it was found that the helix-loop-helix (HLH) protein Id formed protein-protein interactions with three other HLH proteins (MyoD, E12, and E47) and that the heterodimers involving Id were compromised in their ability to bind to the DNA recognition sequences (@benezra_protein_1990).

FD is a protein involved with the floral transition in *Arabidopsis thaliana*.

![**Multiple sequence alignment of the *Arabidopsis thaliana* and *Brassica napus* FD proteins** The indicated regions of the protein are defined as in REF. The heptad structure of the leucine zipper region is displayed below the alignment with the coil interior amino acids and amino acids which are capable of forming electrostatic interactions shaded grey and black respectively.](figuredirectory/brassica_sequence_alignment.pdf){#fig:spring:brassicasequence}

The *Brassica napus* genes showing sequence similarity to the *Arabidopsis thaliana* FD gene have polymorphisms in their coding sequence which may influence their ability to bind DNA.
In a comparative study of FD-like genes from many species, the *Arabidopsis thaliana* FD was found to have four conserved regions (@tsuji_functional_2013): the A region, the LSL region, the bZIP region (made up of a basic and a leucine zipper region) and the SAP region.
Looking at the same regions in *Brassica napus* (Figure @fig:spring:brassicasequence) identifies a number of amino acid changes and deletions in the A region, with there being four different isoforms of the region present in the six *Brassica napus* FD genes.
Comparing the *Brassica napus* FD genes to the *Arabidopsis thaliana* FD gene reveals that, in the A region, the gene copies on chromosomes A8 and C3 show the greatest amino acid sequence similarity to the *Arabidopsis thaliana* gene, with only a single amino acid change present.

![**Protein structure of the *Brassica napus* FD genes complexed with DNA** The protein structure of the CREB protein from REF was mutated to match the amino acids which are present in the *Brassica napus* FD proteins. The green dashed lines indicate hydrogen bonding between atoms. The colour scheme for atoms is as follows: white (carbon), dark blue (nitrogen), yellow (phosphorus), red (oxygen) and light blue (hydrogen).](figuredirectory/protein_structure.pdf){#fig:spring:proteinstructure}

Positions 222 and 223 in the basic region of the bZIP domain differ between the *Brassica napus* FD genes (Figure @fig:spring:brassicasequence).
The crystal structure of the mammalian cAMP responsive element-binding protein (CREB) bZIP transcription factor bound to DNA revealed that position 222 is important as the amino acid sidechain forms a hydrogen bond with the DNA backbone (@schumacher_structure_2000).
bZIP transcription factors are found in all eukaryotes, and this conservation was leveraged by using the crystal structure of the CREB transcription factor to determine the potential effect the polymorphisms observed between the *Brassica napus* FD proteins were having on the DNA binding properties.
Mapping the amino acids in the basic region from the *Brassica napus* FD genes onto the crystal structure of the CREB transcription factor revealed that changing the amino acid in position 222 from an arginine to a histidine disrupts hydrogen bond formation between the protein and the DNA (Figure @fig:spring:proteinstructure).
Having a histidine or an asparagine in the 223 position does not seem to affect the hydrogen bonding in the $\alpha$-helix or between the protein and DNA.

In the SAP region, there are again a number of amino acid changes between the different copies of the *Brassica napus* FD-like genes (Figure @fig:spring:brassicasequence).
Of note is the amino acid polymorphism at position 287 between a theonine and serine.
This position in *Arabidopsis thaliana* becomes phosphorylated and is important for the binding of FD to the protein FT in *Arabidopsis thaliana*, as mutation of the threonine to an alanine disrupts complex formation (REF).
Changing the threonine to a serine was found to not affect FD binding to FT in *Arabidopsis thaliana*, although potentially different kinases are responsible for the phosphorylation of the different residues.

The other amino acid changes and deletions observed between the *Brassica napus* copies of FD may affect the functions of the proteins.
However, there is currently no reported evidence to suggest any of these other polymorphisms are having a functional effect on the proteins.

#### Amino acid differences in the leucine zipper region of the *Brassica napus* FD-like proteins may alter dimerization propensity

![**Helical wheel representation of the homodimers and heterodimer possible with the *Brassica napus* genes XLOC_037807 and XLOC_061874** The blue coloured amino acids have positively charged side chains, the red coloured animo acids have negatively charged side chains and the orange amino acids have polar side chains. The blue and red dotted lines between helical wheels indicate attractive and repulsive electrostatic charges between the two helicies respectively.](figuredirectory/helical_wheels.pdf){#fig:spring:helicalwheels}

The amino acid differences in the leucine zipper region of the *Brassica napus* FD proteins have the potential to alter the dimerization affinity of the proteins.
Previous studies of bZIP transcription factors have revealed that amino acid residues in the `e` and `g` positions of the heptad are important in the determination of dimerization specificity (REF).
Specifically, when the proteins form a coiled-coil structure, the side chain of an amino acid in the `e` position on one $\alpha$-helix is able to form electrostatic bonds with the side chain of an amino acid in the `g` position on the other $\alpha$-helix.
This is illustrated in the helical wheel representations in Figure @fig:spring:helicalwheels.
An example of this, using the positions in Figure @fig:spring:proteinstructure as a reference, is residue 250 (in the `g` position of the heptad) has the capacity to form electrostatic interactions with residue 255 (in the `e` position of the heptad).
Therefore, the charge these residues carry is a factor which determines the dimerization energy between bZIP proteins.
Positions 250, 255, 262 and 271 are all in either the `e` or `g` positions of the heptads and show amino acid polymorphisms which alter the charge of the amino acid side chains.
These polymorphisms suggest that certain dimer combinations of the *Brassica napus* FD genes will be more favoured than others.

![**Heatmap of the dimerization affinity scores computed between *Arabidopsis thaliana* and *Brassica napus* FD leucine zipper regions** The machine learning algorithm discussed in the main text was used to score the dimerization affinity of the leucine zipper regions of the proteins. Higher scoring dimers are dimers more likely to have dimerization energies which are favourable. The leucine zipper regions used for the analysis correspond to the region indicated in Figure @fig:spring:brassicasequence](figuredirectory/fd_dimer_heatmap_without_fus_jun.pdf){#fig:spring:heatmapwofusjun}

![***Same as the last figure but with FOS and JUN included as well. I wanted to show that the difference between FOS and JUN hetero and homodimers was a similar difference as we're seeing with the Brassica napus FD proteins, but that comparison is difficult to make as the overall interaction scores for the FOS and JUN dimers are higher, despite the difference being comparable***](figuredirectory/fd_dimer_heatmap_with_fus_jun.pdf){#fig:spring:heatmapwfusjun}

The sequence analysis suggests that the amino acid polymorphisms observed in the `e` and `g` positions of the heptad may affect the dimerization affinity of the proteins.
To investigate this, a previously published machine learning algorithm (@potapov_data_driven_2015) was used to score the potential interaction affinity of pairs of proteins (Figure @fig:spring:heatmapwofusjun).
The interaction scores range from a minimum of 4.33125 to a maximum of 7.1509, a range of 2.81965, with the higher interaction scores indicating higher affinity binding.
To put these scores into context, the dimerization of the bZIP transcription factors FOS and JUN have been extensively studied in terms of their dimerization affinity (REF).
In particular, the FOS-JUN heterodimer is more thermally stable than either the FOS or the JUN homodimer.
Using the machine learning scoring algorithm of Keating et al., FOS homodimers score 6.24499, JUN homodimers score 6.30238 and FOS-JUN heterodimers score 8.80496.
The score range for FOS and JUN dimers is 2.55997, a similar range as that observed for the *Brassica napus* FD genes, implying that the differences in scores are large enough to suggest a functional effect.
The heatmap (Figure @fig:spring:heatmapwofusjun) illustrates that the six *Brassica napus* FD genes fall into three groups.
XLOC_037807 and XLOC_001134 form a group which have a higher affinity for forming dimers between themselves than with the remaining four proteins.
XLOC_024426 and XLOC_049259 have a mild affinity to form dimers between themselves, but have a higher affinity to form dimers with both XLOC_037807 and XLOC_001134.
Finally, XLOC_033135 and XLOC_061874 have the lowest affinity to form dimers between themselves relative to the other dimers tested, and have the highest affinity to form dimers with both XLOC_037807 and XLOC_001134.
The helical wheel representations (Figure @fig:spring:helicalwheels) show the between $\alpha$-helix electrostatic interactions possible in the homodimers and heterodimer of XLOC_037807 and XLOC_061874.
Both the XLOC_037807 homodimer and the heterodimer have attractive electrostatic interactions between the amino acid side chains in the `e` and `g` positions of the heptad.
The XLOC_061874 homodimer has repulsive interactions between the $\alpha$-helicies, supporting the results in Figure @fig:spring:heatmapwofusjun which suggested a XLOC_037807-XLOC_061874 heterodimer would be more energetically favoured than a XLOC_061874 homodimer.

#### bZIP transcription factors in other organisms reveal that charge influencing amino acid polymorphisms in the `e` and `g` heptad positions are observed in other species

![**Multiple sequence alignment of the *Arabidopsis thaliana* FD protein and its orthologues in *Brassica napus*, *Glycine max*, *Musa acuminata subsp. malaccensis* and *Medicago truncatula*** The alignment is limited to the same regions identified by REF and displayed in Figure @fig:spring:brassicasequence.](figuredirectory/ensembl.pdf){#fig:spring:ensembl}

![**Heatmap of the dimerization affinity scores computed between *Musa acuminata* FD orthologue leucine zipper regions**](figuredirectory/fd_dimer_heatmap_musa_acuminata.pdf){#fig:spring:heatmapmusaacuminata}

To investigate whether polymorphisms influencing dimerization affinity were a common occurance in organisms where gene duplication has occurred, sequences of FD orthologues identified in the Ensembl Plants database were aligned.
Only those species containing multiple FD orthologues in the genome are displayed in Figure @fig:spring:ensembl.
Looking at the leucine-zipper region identifies similar charge influencing polymorphisms in the `e` and `g` heptad positions between the genes within a species.
Charge influencing polymorphisms in the `e` and `g` heptad positions are present in *Glycine max* at positions 360, 362 and 381, *Musa acuminata* at positions 362, 367, 374 and 376 and *Medicago truncatula* at positions 360, 362, 367 and 381.
Using the machine learning algorithm (@potapov_data_driven_2015) method of scoring bZIP transcription factor dimers discussed earlier, the FD orthologues from *Musa acuminata* were tested for their dimerization propensity (@fig:spring:heatmapmusaacuminata).
As with the *Brassica napus* FD genes (@fig:spring:heatmapwofusjun) there are certain dimers which are predicted to be energetically favoured over others.
GSMUA_Achr9P21040_001, for example, is predicted to favourably form dimers with all of the other FD orthologues in *Musa acuminata*, whereas the GSMUA_Achr4P05090_001 homodimer is predcited to be energetically unfavourable compared to the other possible dimers.

![**Multiple sequence alignment of the *Arabidopsis thaliana* FD protein, its orthologues in *Brassica napus* and proteins with high sequence similarity from the *Zea mays* and *Triticum aestivum* proteomes** The alignment is limited to the same regions identified by REF and displayed in Figure @fig:spring:brassicasequence. The *Zea mays* proteins plotted were chosen by selecting the *Zea mays* protein with the highest sequence similarity to the *Arabidopsis thaliana* FD protein and then including the paralogues of the gene as identified in the Ensembl Plants database. The *Triticum aestivum* proteins were identified in the same way, except that in addition to the paralogues, the homeologues of the proteins were also included.](figuredirectory/brassica_wheat_maize_fd.pdf){#fig:spring:wheatmaizefd}

![**Heatmap of the dimerization affinity scores computed for the leucine zipper region between the *Zea mays* protein with the highest protein sequence similarity to the *Arabidopsis thaliana* FD protein and its paralogues**](figuredirectory/fd_dimer_heatmap_zea_mays.pdf){#fig:spring:heatmapzeamays}

![**Heatmap of the dimerization affinity scores computed for the leucine zipper region between the *Triticum aestivum* protein with the highest protein sequence similarity to the *Arabidopsis thaliana* FD protein, its paralogues and the homeologues for all proteins**](figuredirectory/fd_dimer_heatmap_triticum_aestivum.pdf){#fig:spring:heatmaptriticumaestivum}

Likewise, extracting the sequences of the bZIP transcription factor from *Zea mays* and *Triticum aestivum* with the highest protein sequence similarity to *Arabidopsis thaliana* FD, as well as its homeologues, paralogues and the homeologues of the paralogues, and performing multiple sequence alignment reveals charge influencing polymorphisms in the `e` and `g` heptad positions.
At all `e` and `g` positions annotated in Figure @fig:spring:wheatmaizefd, both *Zea mays* and *Triticum aestivum* show charge influencing amino acid polymorphisms.
Interestingly, within the genes from *Triticum aestivum* there are no inter-homeologue polymorphisms which influence the charge of amino acids in heptad positions `e` or `g`.
This suggests that the evolutionary time since polyploidization in wheat has not been sufficient to fix mutations which alter the dimerization affinity of this particular set of bZIP transcription factors.
As with the *Brassica napus* FD proteins and the FD orthologues in other species, the predicted dimerization affinities for the *Zea mays* and *Triticum aestivum* bZIP proteins considered in this study show considerable variation (Figure @fig:spring:heatmapzeamays, Figure @fig:spring:heatmaptriticumaestivum).
For the *Zea mays* bZIP proteins, there are a group of five proteins which are predicted to form dimers with high affinity within the group, with a reduced affinity to form dimers outside of that group (Figure @fig:spring:heatmapzeamays).
For the genes outside of this group, heterodimers are predicted to be energetically favoured compared to homodimers.

#### Computational modelling reveals that the modulation of dimerization affinity for dimerizing transcription factors...

![**test**](figuredirectory/fd_dimerization.pdf){#fig:spring:dimerizationmodelling}

\ch{a + a <=>[7][1] aa}

\ch{a + b <=>[k][1] ab}

\ch{b + b <=>[0.5/4/7][1] bb}

In order to test some of the regulatory repurcussions of altered dimerization, a system of ordinary differential equations were used to model the interactions between two monomers which are able to form homodimers and a heterodimer.
To test how the behaviour of the system depends on the dimerization affinities, three different reaction rates for the homodimerization of the `b` species were tested; 0.5, 4 and 7.
For each of these rates, the heterodimerization rate for the monomers was varied and the steady state concentrations of the various species calculated (Figure @fig:spring:dimerizationmodelling).
For a `b` homodimerization rate of 7, the two homodimer species have the same steady state concentrations at all heterodimerization rates, as expected given they have the same reaction rates.
When all dimerization rates are 7, the steady state concentrations of all dimers are identical.
By setting the `b` homodimerization rate to 0.5, the `bb` homodimer is disfavoured, with an observed increase in the steady state concentration of the undimerizaed `b` monomer.
This has has an interesting interaction with the steady state concentration of the heterodimer.
The steady state concentration of the heterodimer now becomes similar to the steady state concentration of the homodimers at a heterodimer formation rate much much lower than 7.
The implication is that the presence of the `b` monomer favours the formation of the `ab` heterodimer rather than the `aa` homodimer, despite the forward reaction rate of the `aa` homodimer being 2.5 times greater than the `ab` heterodimer formation rate.
Using the *Brassica napus* FD genes XLOC_037807 and XLOC_061874 as an example (Figure @fig:spring:heatmapwofusjun) we see that the XLOC_037807 homodimer is more energetically favoured than the XLOC_037807-XLOC_061874 homodimer, which itself is more energetically favoured than the XLOC_061874 homodimer.
However, given the results of the modelling, the less favourable XLOC_061874 homodimer formation (Figure @fig:spring:helicalwheels) may result in the heterodimer having a similar or greater steady state concentration than the XLOC_037807 homodimer despite the heterodimer being less energetically favoured.
