## Discussion

Whole genome triplication and interspecific hybridization have resulted in extensive gene multiplication within the *Brassica napus* genome.
To investigate the retention of genes in the *Brassica napus* genome, relative to the Arabidopsis genome, a developmental transcriptome was collected.
The decision to focus on regulatory divergence was based on work that suggests that regulatory divergence is one of the primary mechanisms by which genes diverge after a multiplication event[@de_smet_redundancy_2012].
The duplication-degeneration-complementation hypothesis posits that as deleterious mutations occur more frequently than beneficial mutations[@lynch_genetics_1998], that the generation of novel molecular functions to drive gene retention will be a very rare occurrence.
Instead, the hypothesis suggests that mutations that disrupt cis-regulatory elements are gradually fixed by genetic drift, leading to subfunctionalization and therefore retention in the genome of genes present as multiple copies[@force_preservation_1999].
Duplicated gene pairs exhibited extensive divergence in the DNase I footprints surrounding the genes, suggesting that regulatory elements controlling the expression of the duplicated genes have diverged[@arsovski_evolution_2015].
Microarray expression experiments in Arabidopsis revealed that 85% of paralogous regulatory genes exhibit expression that suggests neo- or subfunctionalization[@duarte_expression_2006].
The results presented here agree with these previous studies, in that extensive divergence between the expression of gene copies is observed (Figures \ref{figure:213:allvsexp} and \ref{figure:218:somsimilarity}).
An alternative method of quantifying divergence would be to investigate how the molecular functions of proteins have divergence, such as by assessing the extent to which Arabidopsis null mutants are complemented by homologues from *B. napus*[@tadege_control_2001].
This is, however, a laborious process and would have limited the general trends that can be observed when taking a global approach.

The experimental design of the experiment (section \ref{section:spring:experimentaldesign}) was directed by the focus on flowering time gene divergence following gene multiplication.
Sampling both the apex and the leaf across time allowed for a more complete picture of the floral transition to be obtained.
This is exemplified with the expression traces of flowering time gene homologues in *B. napus*, where leaf specific (Figure \ref{figure:223:ftleaf}), and apex specific (Figures \ref{figure:226:ap1apex}, \ref{figure:230:fdapex}, \ref{figure:231:lfyapex}, and \ref{figure:232:tfl1apex}) expression is observed.
For *BnaSOC1* genes, tissue specific regulatory modules assignment is observed, suggesting that in some tissues two *BnaSOC1* may be behaving similarly and in others they may be divergent.
These regulatory effects would not have been observed had only a single tissue been sampled.
In addition, the observations of expression changes during the cold treatment in the spring variety (Figure \ref{figure:221:go3som}) supports the decision to subject the spring variety to the vernalization treatment, as it allows the effects of growth condition changes to be dissected from expression changes due to vernalization (discussed in section \ref{section:spring:experimentaldesign}).

The retention of genes in the *Brassica napus* genome following gene multiplication events seems to have been driven by functional divergence rather than gene dosage.
Such dosage effects can arise with proteins that form complexes, where a stoichiometric imbalance may result in deleterious phenotypes manifesting.
A prediction of the gene dosage hypothesis states that multiple copies of genes are maintained in the genome following a gene multiplication event through dosage effects.
If gene dosage has driven gene retention in *Brassica napus*, the expectation would be that the multiple copies of genes would be co-expressed.
However, looking across the entire developmental time series, we find some copies of genes are not expressed (Figures \ref{figure:213:allvsexp} and \ref{figure:214:allvsexpflower}) while others display diverged expression traces \ref{figure:218:somsimilarity}).
This suggests that functional divergence, rather than gene dosage, has driven the retention of multiple copies of genes in the *Brassica napus* genome.

Assessing the expression of *Brassica napus* homologues of Arabidopsis flowering time genes allows the extent of conservation between the two species to be determined (section \ref{section:spring:floralintegrators})
For the central floral integrators in *Brassica napus*, there is at least one copy that exhibits an expression trace consistent with the regulatory behaviour of the Arabidopsis homologue.
This suggests a general conservation between the regulatory network underlying flowering in Arabidopsis and *B. napus*, that will aid effors to translate knowledge from Arabidopsis to the crop.

### Limitations of the developmental transcriptome {#section:spring:limitations}

One of the primary limitations of the developmental transcriptome are the restrictions due to the sampling procedure.
Although the dissection of the apical region was adequate to enrich for apically expressed genes (section \ref{section:spring:floralintegrators}), it was not at a scale to resolve more fine grain expression domains.
Controlling the determinacy of the apex in Arabidopsis is mediated between interactions between mutually antagonistic interactions between *AtTFL1*, *AtAP1*, and *AtLFY*[@shannon_genetic_1993].
While all three genes in Arabidopsis increase in expression during the floral transition, they do so within spatially defined regions of the apex.
That the expression increases during the floral transition are observed for all three copies in the apical samples in the *Brassica napus* developmental transcriptome suggests that the samples from the apex represents a pool of these different expression domains.
While this does not limit the use of the data to assess functional divergence, it is an important caveat as the time series does not capture the antagonistic regulatory interactions between these flowering time genes.

The sampling of the apex during development meant that the sampling process was destructive, such that samples could not be taken from the same plants during the time series.
An issue that arose from this was was many plants had to be grown in order to allow for the number of time points sampled.
As the focus on the experiment was the floral transition, the time points were clustered during this period of development to capture the regulatory dynamics in high temporal resolution.
The destructive nature of the sampling resulted in other periods of development not being sampled to the same resolution, one of which was the transition into the cold treatment.
The first time point was taken just before the plants were moved into cold at day 23 post sowing, with the next time point taken being midway through the vernalization treatment at day 43 post sowing.
That 3 weeks passed until the first sample during the cold means that rapid transcriptional changes, such as those that result from the change in photoperiod and ambient growth temperature, cannot be separated from transcriptional changes that occur over longer periods.
