## Chapter summaries

### Floral gene retention and divergence in a spring variety

Whole genome duplication events have occurred throughout angiosperm development[@soltis_polyploidy_2009].
Following whole genome duplication, many genes are expected to be lost due to chromosome rearrangements and fusions[@mandakova_island_2010].
Those genes that remain may partition multiple gene functions between duplicates[@force_preservation_1999; @des_marais_escape_2008], acquire novel functions[@ohno_creation_1970], or retain the same function and be retained in the genome to maintain the correct gene dosage[@birchler_gene_2012; @veitia_gene_2004].
Determining the expression profiles of duplicated genes can provide clues as to which of these processes have led to certain genes being retained in the genome.

In *B.\ napus*, duplicated genes from ancient whole genome duplications[@beilstein_dated_2010; @lysak_chromosome_2005], and more recent polyploidy[@rana_conservation_2004] are present in the genome.
Understanding the genetic pathways controlling flowering in *B.\ napus* would allow for breeding varieties with improved flowering behaviour[@jung_flowering_2009].
However, the combinatorial explosion of regulatory possibilities that result from these duplicated genes complicate efforts to translate knowledge about the floral transition from Arabidopsis where much is known[@srikanth_regulation_2011], to *B.\ napus*.
To elucidate the extent of gene divergence in *B.\ napus*, particularly of the flowering time genes, a transcriptomic time series was conducted and is presented in chapter \ref{chapter:spring}.
While the true extent of gene divergence is difficult to determine without functional data concerning protein activity, taking a genome-wide approach results in more general, widely applicable, conclusions to be drawn.
Such a dataset allowed the changes to the transcriptome to be followed across the floral transition in two tissues, the leaf and apex.
Expressed flowering time genes were found to be retained in the genome at a higher rate than other genes in the genome.
The extensive regulatory divergence observed suggested that other processes, other than gene dosage effects, have contributed to the retention of genes in *B.\ napus*.

The floral transition in Arabidopsis is controlled by a tightly interconnected regulatory network, consisting of multiple feedback loops to result in irreversible, robust flowering[@bouche_florid_2016; @jaeger_interlocking_2013].
At least one *B.\ napus* homologue, and often more, exhibited expression profiles consistent with the homologous gene in Arabidopsis, suggesting a general conservation between the expression domains of the genes in Arabidopsis and *B.\ napus*.
A dramatic pattern of tissue specific expression of homologues was observed for *BnSOC1* genes, suggesting a partitioning of spatial expression domains between different homologues.
The expression profiles also suggest that different *BnSOC1* genes have different sensitivities to environmental signals.
Given the role *SOC1* plays in integrating multiple environmental signals in Arabidopsis[@lee_regulation_2010], the work presented here suggests that *BnSOC1* genes are an almost archetypal example of the ways subfunctionalization can manifest.

Changes to cis-regulatory elements represent a way by which the expression of genes can diverge[@force_preservation_1999].
The expression profiles of *BnTFL1* were shown to correlate with the presence and absence of regions of sequence conservation with Arabidopsis *TFL1* downstream of the gene.
These regions of sequence conservation occur in areas of downstream sequence identified as cis-regulatory elements in Arabidopsis[@serrano_mislata_separate_2016].
The similarities between the expression profiles of particular *BnTFL1* genes and the expression domains the regulatory elements defined in Arabidopsis suggest conservation between how the *BnTFL1* genes and the *TFL1* in Arabidopsis are regulated.
Although correlative, this analysis suggests that cis-regulatory element changes may represent an important driver of subfunctionalization in *B.\ napus*.
It also highlights how studies dissecting regulatory elements in a model species can lead to insights in a crop.
As the regulatory elements controlling other genes are elucidated to the level of *TFL1*[@serrano_mislata_separate_2016] it will be interesting to investigate whether the other cases of regulatory divergence observed in *B.\ napus* flowering time genes can be explained with regulatory element changes.

The results from analysis of *BnFD* sequence divergence demonstrate that determining gene divergence from expression data leads to an underestimation of the true divergence present.
Sequence differences between *BnFD* homologues are predicted to alter dimerization affinities of BnFD proteins, resulting in certain BnFD dimers being more likely to occur than others.
This may be a common method of bZIP divergence, as similar differences between FD-like proteins in a range of plant species are observed.
A simple computational model was used to understand the consequences of different dimerization affinities between BnFD proteins.
The simulations highlighted that novel regulatory behaviours are possible as a result of different dimerization affinities.
While the results are currently only theoretical, dimerization has been shown to facilitate gene regulatory logic[@amoutzias_choose_2008] and is a factor influencing the evolution of bZIP transcription factors[@amoutzias_one_2007].
However, without further data it is difficult to conclude whether the observed differences are biologically relevant.
This would require determining if different BnFD dimers possess different activities, such as different preferences in binding sites.
The changes observed may also represent a form of complementary change, whereby BnFD proteins are diverging simultaneously with binding partners.
As data on FD protein interactions become available it will be interesting to revisit these results.

The questions of gene retention raised in this chapter have to be viewed in the context of *B.\ napus* being a crop, grown under artificial selection.
Although gene redundancy is not necessarily stable in natural conditions[@nowak_evolution_1997; @lynch_evolutionary_2000], it may be selected for in an agricultural setting where consistency is paramount.
It is suggested that polyploidy represents a method of fixing heterosis, or hybrid vigour, in a crop[@dubcovsky_genome_2007; @comai_advantages_2005; @paterson_polyploidy_2005; @yao_genomic_2013].
The regulatory divergence observed suggests that polyploidy may indeed lead to a 'Swiss Army knife' of similar genes being retained in the genome, each adapted to particular growth conditions, tissue, or stage of development, which can be expressed as and when it is needed.

### Effects of a requirement for cold on regulatory divergence

An important agronomic trait of *B.\ napus* is whether the plant requires a period of cold in order to flower.
Varieties that do not require cold are often grown in Canada and Northern Europe, where harsh winters would damage crops grown during winter, whereas varieties that do require cold in order to flower are grown in Europe and Asia[@friedt_oilseed_2009].
This requirement for cold is called vernalization, and is a pathway that is well understood at the molecular level in the model species Arabidopsis[@song_remembering_2013].
The pathway is arguably the most well understood flowering time pathway in *Brassica* crops with an array of different studies finding *B.\ napus* vernalization gene homologues associated with flowering and exhibiting sequence divergence[@zou_comparative_2012; @irwin_functional_2012; @schranz_characterization_2002].
How a requirement for cold influences the overall transcriptome, however, and whether the *BnFLC* genes influence the expression of certain floral integrators more than others was not known.

In chapter \ref{chapter:winter} the effects of a requirement for cold on the transcriptome of *B.\ napus* were assessed by comparing a winter variety, Tapidor, and spring variety, Westar.
The potential importance of the leaf during vernalization in *B.\ napus* was revealed through an expansion of expressed gene number in Tapidor relative to Westar.
As the action of *FLC*, a key vernalization sensitive gene, acts at both the leaf and the apex[@searle_transcription_2006], exploring the biological significance of this increased gene set expressed in the leaf in the winter variety will be a central question motivating future work.
Correlation analysis suggested that different factors influence the transcriptome depending on the tissue, with the leaf seemingly influenced by plant age and the apex by developmental stage.
This is counter to expectations from Arabidopsis[@levey_natural_2005], but may represent an instance of artificial selection for leaf senescence to allow metabolites to be remobilized from leaves to the growing flowers and seeds, leading to yield increases.
Taken together, these findings suggest that the vernalization response may be affecting both the signals the leaf transmits to the apex and the way the apex interprets those signals, consistent with the role of *FLC* in Arabidopsis[@searle_transcription_2006].

Considering genes involved with the vernalization response, copies of *BnFLC* were found to exhibit varietal differences and expression profiles consistent with these genes mediating the vernalization response in the winter variety.
The two best candidates for conferring the vernalization response in Tapidor, based on their expression profiles, are *BnFLC.A2* and *BnFLC.A10*, consistent with previous studies[@schranz_characterization_2002].
Not all *BnFLC* genes were found to respond similarly to cold.
*BnFLC.A3b* and *BnFLC.C2* are not stably silenced in Tapidor, revealing that expression of these genes does not prevent flowering.
Potentially these *BnFLC* genes may require longer periods of cold to become fully repressed, in a similar manner to certain *FLC* alleles in Arabidopsis[@coustham_quantitative_2012; @duncan_seasonal_2015].
The experimental design decision to subject a spring variety to vernalization may initially seem strange.
However, doing so allowed candidate *BnFLC* genes for the mild vernalization response in Westar to be identified[@murphy_vernalization_1994].
The theoretical consequences of having differently tuned *FLC* homologues were considered, and proposed to allow plants to disentangle the length of vernalization and the temperatures experienced during cold.
Other genes involved with the vernalization response exhibited differences in the magnitude of expression between varieties.
However, the consequences of these observed differences are difficult to assess.

The expression differences of *BnFT* and *BnSOC1* genes between varieties were consistent with the effects of *BnFLC* mediated repression, in line with findings from Arabidopsis[@lee_agamous_like_2000; @michaels_integration_2005].
Two genes exhibited very different expression profiles in Tapidor relative to Westar.
*BnFD.A1* and *BnAP1.A2* exhibited expression divergence in the spring variety, but the expression profiles of these genes in Tapidor were more consistent with the other homologues of those genes.
Determining whether these differences influence the vernalization response, or represent differences due to variety, would require a more thorough assessment of transcriptomic changes involving multiple winter and spring lines.
Either way, given the results from chapter \ref{chapter:spring}, the altered expression of a single *BnFD* may have large impacts on the BnFD dimers observed.

### Data dissemination using a web application

An important part of the scientific process is the sharing of data.
Sharing data allows others in the field to more readily consider their results in light of previous studies.
This is of particular relevance to extremely large transcriptomic datasets, where the scale of the data makes it infeasible for every gene to be investigated by a single group of researchers.
The ready availability of large datasets such as this allow for a division of labour, with insights on particular genes made by experts.

The transcriptomic time series presented in this work is of general interest to any Brassica researcher investigating genes expressed during the floral transition.
In chapter \ref{chapter:website} a web resource that facilitates access to the dataset is described.
The search features allow researchers to find genes of interest and plot expression profiles in an intuitive manner.
To ensure the resource is as generally useful as possible, the database structure and plotting features facilitate the easy inclusion of additional data.
