## Concluding thoughts

The original aim of this project was to determine the extent to which the regulatory network underlying flowering in Arabidopsis could be applied to *B. napus*.
The intention was to use the transcriptomic time series to reduce the complexity of the network by grouping similarly expressed flowering time gene homologues together as a single network node.
The work presented in this thesis, however, revealed that regulatory divergence between homologous genes is frequently observed in *B.\ napus*.
This introduced the challenge of how to deal with the combinatorial explosion of regulatory possibilities and to reduce the model to a computationally tractable system.
The obstacle of additional regulatory complexity caused by multiple gene copies changed the direction of the project to instead investigate how the dynamics of these genes have diverged.

This study represents the first study in *B.\ napus* to follow the transcriptome before, during, and after vernalization.
Arabidopsis floral genes were found to be retained in the genome more frequently than expected, with the patterns of regulation suggesting different selective pressures are acting on the genes.
Analysis of both the leaf and apex transcriptomes revealed that these tissues are distinct in their transcriptional responses, and identified cases where floral gene homologues have diverged in terms of their spatial expression domains.
The importance of cis-regulatory elements in the evolution of duplicated genes is highlighted, and represents an example of how research in model species can begin to be translated to a crop species.
The findings that similarly expressed genes exhibit functionally relevant sequence differences calls into question the very assumption on which the original project aim was based, namely, that similarly expressed genes can be considered as a single node in the network.

Despite the value of this work in elucidating regulatory divergence between gene homologues, a key question remains: how much closer are we to a regulatory network of the *B.\ napus* floral transition?
This project emphasizes the problems inherent to determining a simple gene regulatory network of a crop that has experienced multiple rounds of gene duplication.
Instead, the complexity of polyploid networks could be approximated by sub-networks based on modules with little regulatory dependence.
This subsetting will require a better understanding of how the multiple copies have diverged, both in gene expression and protein activity, and provides a clear direction for future work in *B.\ napus*.
