## *Arabidopsis thaliana* as a model for flowering time {#section:intro:arabidopsis}

Model species have been key to the progression of biology by allowing researchers from all over the world to collaborate and focus research effort on common systems[@flavell_role_2009].
Although it has been worked on since the turn of the 20^th^ century, it was not until the 1970s, and the desire for a plant well-suited to molecular genetics, that *Arabidopsis thaliana* (hereafter Arabidopsis) cemented its position as *the* model plant species[@koornneef_development_2010; @redei_arabidopsis_1975].
Arabidopsis makes a good model organism due to a short generation time, a small physical size, and because it produces many seeds from a single, self-pollinated flower.
Experimental tools have been developed to facilitate both forward genetics (identifying genotype from phenotype) and reverse genetics (identifying phenotype from genotype).
The use of ethyl methanesulphonate to mutagenize Arabidopsis facilitated forward genetic screens to identify plants that are deficient in a phenotype of interest[@kim_ems_2006].
Such screens allowed the identification of global regulators of floral organ identity[@weigel_abcs_1994].
For reverse genetics, transformation methods using *Agrobacterium tumefaciens* were developed in the 1980s, allowing laboratory made genetic constructs to be inserted into the plant[@nester_agrobacterium_2015].
Another factor in the use of Arabidopsis as a model was the availability of a complete genome sequence, which was the first plant genome fully sequenced[@the_arabidopsis_genome_initiative_analysis_2000], and the third multicellular organism after *Caenorhabditis elegans*[@the_celegans_sequencing_consortium_genome_1998] and *Drosophila melanogaster*[@adams_genome_2000].
This was in part possible due to the relatively small size of the Arabidopsis genome, which in hindsight also contributed to the success of mutant screens.
The availability of these tools for manipulating the genome of Arabidopsis have allowed multiple developmental pathways to be dissected in the plant.

One developmental pathway of particular interest is the transition from vegetative growth to reproductive growth[@andres_genetic_2012].
Timing this transition correctly is extremely important to ensure reproductive success of plants growing in the wild and maximal yields of plants grown as crops.
The presence of the above mentioned genetic tools has allowed a deep understanding of the floral transition in Arabidopsis to be attained.
Multiple pathways sense a myriad of internal and external cues to ensure that flowering in the plant is properly timed.
The variation in floral response between different Arabidopsis accessions has also aided this work, making use of association studies to identify genes that influence the floral response most strongly[@boss_multiple_2004].
There are five main pathways that influence flowering in Arabidopsis.
These are the photoperiod pathway, the autonomous pathway, the vernalization pathway, the hormone pathway, and the ageing pathway[@srikanth_regulation_2011].
All of these pathways converge and are integrated by a central network of genes to ensure that the plant flowers at an optimal time.
In this section current knowledge of each of the pathways, and the key genes involved in them, will be summarized.

### Floral pathways

The floral pathways can be divided into whether they respond to external (exogenous) or internal (endogenous) cues.
The pathways that sense exogenous cues (the photoperiod and vernalization pathways) will be considered first.

The photoperiod pathway allows the plant to sense the day length.
This is achieved through close association of the plant's circadian clock and light sensing apparatus.
The circadian clock is a regulatory network that maintains a consistent oscillatory signal in the plant[@mcclung_plant_2006].
*CONSTANS* (*CO*) encodes a zinc finger transcription factor whose expression is downstream of the circadian clock, with *CO* mRNA accumulating and degrading in a regular manner each day[@an_constans_2004; @suarez_lopez_constans_2001].
However, CO protein is only able to accumulate when the plant is exposed to light, as it is rapidly degraded during the night[@valverde_photoreceptor_2004].
During short days, *CO* mRNA accumulates.
CO protein is translated, but is rapidly degraded and cannot accumulate.
However, during long days, *CO* mRNA is expressed at dusk, allowing CO protein to accumulate.
*CO* accelerates the floral transition by binding to the promoter and activating the expression of a floral activator called *FLOWERING LOCUS T* (*FT*)[@samach_distinct_2000; @yoo_constans_2005; @kobayashi_pair_1999; @tiwari_flowering_2010].
This allows Arabidopsis to sense the day length and flower when the days are long enough.

Arabidopsis plants are capable of exhibiting two main life strategies[@shindo_natural_2007; @koornneef_naturally_2004].
Summer annual accessions germinate in spring, flower in the summer, and are able to set seed before winter.
This is possible in warmer climates, such as central and southern Europe, where the length of summer is long, but in temperate climates, such as northern Europe, a winter annual strategy is followed[@thompson_spatiotemporal_1994].
These plants germinate in late summer or autumn, remain vegetative during the winter, and flower in the spring.
If a plant following a winter annual strategy were to rely solely on the photoperiod pathway to determine flowering time, there is a risk that the day length in autumn would be long enough to activate flowering.
This would result in vastly reduced reproductive success for the plant, due to the seed filling period taking place during the photosynthetically poor winter months.
The vernalization pathway ensures that winter annual plants remain vegetative until after a period of cold has been experienced by the plant[@song_remembering_2013].
The vernalization response was found to be largely determined by two genes; *FRIGIDA* (*FRI*) and *FLOWERING LOCUS C* (*FLC*)[@shindo_role_2005].
*FLC*, a MADS-box containing transcription factor[@michaels_flowering_1999], acts antagonistically to CO by binding to the first intron of the *FT* locus[@helliwell_arabidopsis_2006; @searle_transcription_2006] and repressing *FT* gene expression.
In winter annual lines *FLC* is high when the plant germinates.
During cold conditions the expression of *FLC* decreases.
This repression is mitotically stable.
What this means is that a plant that has experienced cold, when returned to growth in warm conditions, will continue to exhibit low *FLC* expression.
In this manner *FLC* expression acts as a memory of whether a plant has experienced winter.
This temporal separation of when the signal is sensed and when it is responded to is possible through epigenetic changes at the *FLC* locus.
Cold treatment results in the expression of *VERNALIZATION INSENSITIVE 3* (*VIN3*), which in turn recruits the Polycomb Repressive Complex 2 (PRC2) to the *FLC* locus.
PRC2 changes how the *FLC* DNA is packed in the nucleus, silencing it in a mitotically stable manner.
In summer annual accessions of Arabidopsis the expression of *FLC* is low when the plant germinates, negating the requirement for cold.
Whether a plant is a winter or spring accession is largely dependent on the gene *FRI*.
*FRI* activates *FLC* expression by stimulating the activity of a histone methyltransferase called *EARLY FLOWERING IN SHORT DAYS*[@hyun_arabidopsis_2017; @kim_establishment_2005].
An active allele of both *FRI* and *FLC* is therefore required for a plant to exhibit a vernalization response.

The other floral pathways sense endogenous cues in the plant.
The autonomous pathway was named after a collection of mutant lines that flowered late regardless of the photoperiod the plants were grown under; *LUMINIDEPENDENS* (*LD*), *FCA*, *FY*, *FPA*, *FLOWERING LOCUS D* (*FLD*), *FVE*, *FLK*, and *RELATIVE OF EARLY FLOWERING 6* (*REF6*)[@simpson_autonomous_2004].
However, the late flowering phenotype of these mutants was no longer observed if the plants were vernalized, suggesting that the autonomous and vernalization pathways converge[@koornneef_genetic_1991].
This convergence was found to occur at *FLC*, with *FRI* and the autonomous pathway activating and repressing *FLC*, respectively, to set intial levels of the gene.
In this manner, *FRI* and the autonomous pathway together control the vernalization response of the plant, with the autonomous pathway required in spring annual accessions to repress *FLC* expression[@amasino_timing_2010].

A class of plant hormones called the gibberellins (GA) have been found to be linked to the floral transition, although other classes of plant hormones have also been implicated[@davis_integrating_2009].
Plants that are mutant in the synthesis of GAs have a severe delay in flowering during short days, but show little effect during long days[@wilson_gibberellin_1992].
This indicates that the GA pathway in Arabidopsis is mainly involved with promoting flowering during non-inductive conditions.

Finally, the ageing pathway represses flowering when the plant is juvenile and promotes it when the plant ages.
This response is mediated by microRNAs, 18 to 24 nucleotide RNA molecules that do not encode proteins[@spanudakis_role_2014].
These small molecules are involved with controlling the regulation of genes across both plant and animal kingdoms.
With regard to the floral transition, two families of microRNA are particularly important; *miR156* and *miR172*[@yamaguchi_regulation_2012].
The *miR156* family is expressed in the juvenile phase and decreases in expression as the plant ages.
Conversely, the *miR172* family accumulates in expression as the plant ages.
The *miR156* family targets the *SQUAMOSA PROMOTER BINDING-LIKE* (*SPL*) transcription factors, repressing their expression.
The SPL transcription factors activate the expression of a number of floral activators, namely *FT*, *SUPPRESSOR OF CONSTANS 1* (*SOC1*), *APETALA 1* (*AP1*), and *LEAFY* (*LFY*).
Therefore the decrease in expression of the *miR156* family as the plant ages allows these floral activators to be expressed.
Another SPL transcription factor target is *miR172*.
Hence, the decrease in *miR156* expression results in the increase of *miR172* expression.
*miR172* represses the activity of the Arabidopsis *APETALA 2* (*AP2*) family of genes, a set of floral repressors that have found to have binding sites upstream of floral activators.
The feedback loop created ensures that the switch from the juvenile to the mature growth phase is stable.
The regulation of *miR156* is hypothesized to be regulated by sugar or carbohydrate availability, which is used as a proxy for the age of the plant.

### Floral integrators {#section:intro:floralintegrators}

\begin{figure}[htbp]
\includegraphics{figuredirectory/gene_regulatory_network.pdf}
\caption{{The core network of floral integrators.}}{Many
regulatory interactions have been found between the nine floral
integrators depicted here. This results in a tightly interconnected gene
regulatory network, with many possible feedback loops and control
mechanisms. Only key regulatory and protein-protein interactions are depicted here. Adapted from Bouch√© et
al. (2016)\textsuperscript{296}.}\label{figure:1xx:floralnetwork}
\end{figure}

The pathways described above converge onto a set of floral integrator genes, that mediate the transition to flowering.
The core of this network is composed of relatively few transcription factors with multiple regulatory links between them.
These feedback loops allow for the signals from the flowering time pathways to be appropriately interpreted and provide robustness to the system[@jaeger_interlocking_2013].

Both the photoperiod and vernalization pathways converge onto the expression of the floral activator *FT*.
Grafting experiments in a number of plant species led to the conclusion that a floral inducer, referred to as the florigen, was transported from leaves to the shoot apex to initiate flowering[@aksenova_florigen_2006; @lang_promotion_1977].
It later emerged that the florigen, initially hypothesised to be a plant hormone, was the protein FT.
*FT* is expressed in the phloem companion cells, and the FT protein is transported in the plant vasculature from leaves to the apex to promote flowering[@jaeger_ft_2007; @mathieu_export_2007; @notaguchi_long_distance_2008].
The gene was identified from a photoperiod sensitive mutant plant that exhibited delayed flowering when the plants were grown in long days[@koornneef_genetic_1991].
This photoperiod sensitivity was found to be due to *FT* being directly regulated by the circadian clock gene *CO*[@samach_distinct_2000; @yoo_constans_2005; @kobayashi_pair_1999].
The vernalization pathway also influences the expression of *FT*, with *FLC* binding to a site within the first intron of *FT* to repress its expression[@helliwell_arabidopsis_2006; @searle_transcription_2006].
*FT* activates the expression of three MADS-box containing proteins that promote flowering; *FRUITFULL* (*FUL*)[@wigge_integration_2005], *SOC1*[@moon_analysis_2005], and *AP1*[@abe_fd_2005].
These will be discussed in more detail later in this section.

A gene found to act in an antagonistic manner to *FT* in determining the floral transition is *TERMINAL FLOWER 1* (*TFL1*).
Wild type Arabidopsis flowers develop in an indeterminate manner[@alvarez_terminal_1992].
When the transition to flowering occurs, the vegetative meristem converts into an inflorescence meristem that in turn generates the floral structure.
Additional inflorescence meristems, and eventually floral meristems, develop on the side of the main inflorescence stem.
However, the shoot apical meristem, located at the top of the floral structure, remains as an inflorescence meristem, and hence floral growth is indeterminate in Arabidopsis.
Mutants in *TFL1* result in the primary inflorescence meristem converting into a floral meristem, such that the floral structure terminates in a flower as opposed to maintaining an indeterminate state[@shannon_mutation_1991].
In addition, *TFL1* null mutant plants also undergo the floral transition earlier than wild type plants[@bradley_inflorescence_1997].
*TFL1*, therefore, is a repressor of the floral state, influencing meristem identity and regulating the timing of the floral transition.
The inflorescence meristem identity is maintained by TFL1 protein through limiting the activity of *AP1* and *LFY*[@shannon_genetic_1993; @gustafson_brown_regulation_1994; @ratcliffe_separation_1999].
In addition to transcriptional repression, TFL1 protein limits the activity of AP1 and LFY protein, as shown by Arabidopsis lines that overexpressed *TFL1* and either *AP1* or *LFY*[@ratcliffe_separation_1999].
Likewise, AP1 and LFY repress *TFL1* gene expression, with the mutual antagonism likely leading to the sharp expression boundaries required to accurately specify floral development[@shannon_genetic_1993; @liljegren_interactions_1999]
*TFL1* and *FT* are very closely related proteins, with only 39 amino acid changes that distinguish the two proteins[@ho_structural_2014].
Indeed, mutations have been found that produce TFL1 proteins that are FT-like and vice versa [@ahn_divergent_2006; @hanzawa_single_2005; @ho_structural_2014].

Despite being central floral integrators, both *FT* and *TFL1* do not possess DNA binding activity themselves and are therefore not transcription factors.
Instead, the proteins of both genes interact with the FD protein, a bZIP transcription factor[@jaeger_interlocking_2013; @abe_fd_2005; @wigge_integration_2005].
*FD* was originally identified as a late flowering mutant[@koornneef_genetic_1991] found to repress the phenotype of *FT* overexpression lines[@abe_fd_2005].
The FD protein was confirmed as an interacting partner of FT in a yeast two-hybrid screen[@wigge_integration_2005], and was found to also bind FT *in vitro*[@abe_fd_2005].
Two lines of evidence point towards FT and FD interacting to promote *AP1* expression.
The first is the ectopic expression of *AP1* observed in *FD* overexpression lines that is dependent on the presence of *FT*[@abe_fd_2005].
The second line of evidence are chromatin immunoprecipitation experiments conducted in an *FD* overexpression line.
Antibodies for the FT protein were used to enrich DNA and *AP1* promoter sequence was found in a *FT* dependent manner[@wigge_integration_2005].

A homeotic mutation in Arabidopsis that severely impacts the transition from vegetative to floral growth is in the *LEAFY* (*LFY*) gene.
*LFY* was identified in a mutant screen as a mutant that produced leafy shoots in the place of flowers, with the flowers that were produced often lacking petals and stamens[@schultz_leafy_1991].
The gene was found to play a role both in the transition to flowering, but also in specifying the determinacy of the floral meristem[@huala_leafy_1992].
LFY binds to DNA as a dimer, with the cooperative nature of this binding suggested to facilitate a sharp developmental transition[@hames_structural_2008].
*LFY* has been found to regulate or interact with a number of other genes involved with the floral transition.
Increasing *LFY* expression precedes an increase in *AP1* expression[@hempel_floral_1997], with additional evidence suggesting that *AP1* is a direct target of *LFY*[@wagner_transcriptional_1999; @william_genomic_2004].
Other genes important for flowering that are regulated by *LFY* are *TFL1*[@weigel_developmental_1995], *AGAMOUS* (*AG*)[@weigel_activation_1993; @hong_regulatory_2003], and *CAULIFLOWER* (*CAL*)[@william_genomic_2004], with *LFY* itself being regulated by *SOC1* and *AGAMOUS-LIKE 24* (*AGL24*)[@lee_soc1_2008].
In addition, a suite of transcription factors and signalling molecules, both related to flowering time and not, were found to respond to *LFY* activation or have *LFY* binding detected in promoter regions[@william_genomic_2004; @winter_leafy_2011].
Interactions between *LFY* and the photoperiod pathway[huala_leafy_1992; @blazquez_leafy_1997], and the GA pathway[@okamuro_flowers_1996; @eriksson_ga4_2006], suggest that many environmental pathways that regulate flowering converge on *LFY*, underpinning its role as a floral integrator.

*AP1* is a MADS-box containing transcription factor[@alejandra_mandel_molecular_1992] important for both controlling meristem identity and floral organ specification.
Null mutations in the *AP1* gene result in the mutant plants lacking petals[@koornneeff_ems_1982], a consequence of the role *AP1* has in specifying floral organ identity.
Additionally, the sepals that usually surround flowers in *AP1* mutant plants are instead converted to bracts, with secondary flower buds formed in the axils of each bract[@irish_function_1990].
This particular phenotype suggests that *AP1* is important for the conversion of the inflorescence meristem into a floral meristem, as without an active version of the *AP1* protein the floral meristem partially reverts back to an inflorescence meristem[@alejandra_mandel_molecular_1992].
This is also supported by the *AP1* overexpression phenotype, where apical and lateral shoots are converted into flowers[@mandel_gene_1995].
The modulation of meristem activity by *AP1* is believed to be via the plant hormone cytokinin, with *AP1* affecting both the biosynthesis and degradation pathways of the hormone[@han_cytokinin_2014].
25% of the putative targets of *AP1* are other transcription factors, such as *LFY*, explaining why plants mutant in and overexpressing *AP1* have such dramatic effects on flower development in Arabidopsis[@kaufmann_orchestration_2010].
*AP1* and *LFY* double null mutants had a significantly more severe phenotype than either of the single mutants, indicating that these genes seem to act synergistically[@weigel_leafy_1992].
In mutant plants lacking *AP1*, *SHORT VEGETATIVE PHASE* (*SVP*), *AGL24*, and *SOC1* become ectopically expressed[@liu_specification_2007], with further evidence suggesting that *AP1* directly represses the expression of these genes[@gregis_agamous_like24_2008].
*SVP* and *AGL24* maintain the vegetative and inflorescence meristems respectively[@liu_specification_2007].
The expression of *AP1*, therefore, confers a floral state to the meristem.

A gene involved with integration of inputs from an array of different flowering time pathways is *SOC1*[@lee_regulation_2010].
The gene was discovered[@onouchi_mutagenesis_2000] and rediscovered[@lee_agamous_like_2000] in Arabidopsis through a number of different experimental methods.
The *SOC1* gene was found to be differentially expressed after activation of an inducible CO protein in the absence of protein translation, suggesting *SOC1* is a direct target of CO[@samach_distinct_2000] and thus downstream of the photoperiod pathway.
Indeed, *SOC1* gets its name as a mutant in the *SOC1* gene was able to suppress the early flowering phenotype of an Arabidopsis line overexpressing *CO*[@onouchi_mutagenesis_2000].
The overexpression of *SOC1* in a vernalization requiring line of Arabidopsis was able to overcome the vernalization requirement, suggesting that *SOC1* is also a part of the vernalization pathway[@lee_agamous_like_2000].
Subsequent analysis has revealed that this regulation is likely to be direct, as a transcription factor motif in the *SOC1* promoter was found to be bound by FLC *in vitro* and required for *SOC1* repression *in vivo*[@helliwell_arabidopsis_2006; @hepworth_antagonistic_2002].
*SOC1* was initially discovered, therefore, as acting downstream of the vernalization and photoperiod flowering pathways, and subsequent investigations have revealed that *SOC1* is involved with additional floral pathways.
The rescue of a GA biosynthesis mutant with the treatment of GA causes an increase in the expression of *SOC1*[@moon_soc1_2003].
This finding, in addition to the *SOC1* mutant being less sensitive to the treatment of GA[@moon_soc1_2003], suggests *SOC1* integrates the response to the GA-dependent, hormonal pathway.
*SOC1* has also been implicated in the intermittent cold-sensing pathway[@seo_crosstalk_2009] and the age-dependent flowering pathway[@wang_mir156_2009].
All this evidence points towards *SOC1* being a central integrator that is the convergence point of a range of flowering time control pathways.

The regulation of *SOC1* is tied to another MADS-box containing flowering time gene, *AGL24*, as both regulate each other in a positive feedback loop[@liu_direct_2008].
The AGL24 protein was found to be important for the entry of SOC1 protein into the nucleus, with AGL24 and SOC1 binding as a probable dimer at the promoter of *LFY*[@lee_soc1_2008].
*AGL24* seems to act somewhat redundantly with *AP1* and *SVP* to repress certain genes involved with floral organ specification to properly pattern the developing flower[@gregis_arabidopsis_2009].

The *SOC1* gene is at least somewhat redundant with the gene *FUL*, suggesting that *FUL*[@melzer_flowering_time_2008], like *SOC1*, is activated by *FT* expression[@teper_bamnolker_flowering_2005].
The gene was characterised as affecting the development of the Arabidopsis seed pod[@gu_fruitfull_1998], but was also found to act earlier in the reproductive phase by controlling flowering time and meristem identity alongside *AP1* and *SOC1*[@ferrandiz_redundant_2000].
Indeed, plants that are mutant in both *FUL* and *SOC1* remain vegetative, and almost resemble perennial plants[@melzer_flowering_time_2008].
SOC1 and FUL interact[@folter_comprehensive_2005], and have been found to bind to and activate *LFY* expression[@balanza_sequential_2014].

Finally, *SVP* is a gene that seems to have a dual role as a floral repressor early in development, and as a floral meristem identity specification gene later in development, with differing target genes[@gregis_identification_2013].
As a floral repressor, it has been found to form a heterodimer with FLC, although lack of SVP does not significantly impact the targets of FLC.
This is not mutual, however, as the presence of FLC causes a large effect on the targets of SVP, with the number of targets doubling[@mateos_combinatorial_2015].
As with FLC, SVP has been found to bind at the *FT* locus to delay the floral transition[@jang_genetic_2009].
When the floral transition occurs, however, SVP seems to act redundantly with AP1, AGL24, and SOC1 to maintain an indeterminate meristem[@immink_characterization_2012; @liu_regulation_2009; @gregis_arabidopsis_2009; @gregis_agamous_like24_2008].
Extensive heterodimer formation has been demonstrated between the MADS-box containing flowering time proteins[@folter_comprehensive_2005].
It therefore seems likely that the role of SVP changes depending on which proteins it dimerizes with[@gregis_agamous_like24_2008; @mateos_combinatorial_2015].

Being sessile organisms, plants need to interpret environmental cues and respond appropriately.
The different floral pathways allow for these environmental cues, and for endogenous cues such as age, to be interpreted.
The combined interactions of the floral integrators discussed here allow for these signals to be integrated, providing robustness to the floral transition[@jaeger_interlocking_2013].
The flowering time genes and pathways identified in Arabidopsis have been found to be somewhat conserved in a wide range of crop species[@jung_flowering_2009], leading some to dub Arabidopsis the 'Rosetta stone' of flowering time research[@simpson_arabidopsis_2002].
