## The origin of *Brassica napus* and why flowering time is important {#section:intro:brassica}

The *Brassica* genus is in the same taxonomic family as Arabidopsis, the Brassicaceae[@al_shehbaz_generic_2012], and comprises a large number of economically important vegetable and oil crops that show broad morphological divergence.
Among the *Brassicas* are both diploid and tetraploid species.
Diploid species of the *Brassica* genus include *B. rapa* (Chinese cabbage, turnip, and pak choi), *B. oleracea* (kale, cabbage, broccoli, cauliflower, and Brussels sprout), and *B. nigra* (black mustard).
A theory proposed by Woo Jan-choon in 1935, that has become known as the triangle of U, posits that ancestors of the above diploid species hybridized to give ancestors of the tetraploid species of *Brassicas*.
These tetraploid species are *B. napus* (oilseed rape, swede, kale), *B. carinata* (Ethiopian mustard), and *B. juncea* (Indian mustard).
As the tetraploids are the result of interspecies hybridization events they are termed allopolyploids.
Progenitors to modern day *B. rapa* and *B. oleracea* plants are thought to have hybridized to form ancestral *B. napus* less than 10,000 years ago[@rana_conservation_2004], with multiple hybridizations having taken place to give the modern *B. napus* gene pool[@allender_origins_2010].
Rapeseed crops such as *B. napus*, are the second most cropped oil crop worldwide comprising 13% of the total yield[@fao_2017], with the oil being used as a vegetable oil and for industrial lubricants.
In the UK, 13% of the total area on which crops were grown in 2016 (608,000 hectares) were used for oilseed crops, generating Â£541 million in income[@agriculture_uk_2017].
Wheat grown following oilseed rape will yield 10% more than a wheat grown continuously, on average[@christen_effect_1993].

\begin{table}[htp]
\centering
\resizebox{\textwidth}{!}{%
\begin{tabular}{llll}
\toprule
Species & Group & Common name & Organ consumed \\
\midrule
\emph{Brassica oleracea} & \emph{acephala}          & Kale, collards           & Leaves \\
                         & \emph{capitata capitata} & Cabbage                  & Terminal leaf buds (heads) \\
                         & \emph{capitata sabauda}  & Savoy cabbage            & Terminal leaf buds (heads) \\
                         & \emph{costata}           & Tronchuda cabbage        & Loose heads \\
                         & \emph{gemmifera}         & Brussels sprouts         & Vegetative buds \\
                         & \emph{botrytis botrytis} & Cauliflower              & Inflorescences \\
                         & \emph{botrytis italica}  & Broccoli                 & Inflorescences \\
                         & \emph{gongylodes}        & Kohlrabi                 & Stem \\
                         & \emph{albogabra}         & Chinese kale             & Leaves \\
\emph{Brassica rapa}     & \emph{rapa}              & Turnip                   & Roots \\
                         & \emph{rapa}              & Turnip greens            & Leaves \\
                         & \emph{rapa}              & Turnip tops              & Shoots \\
                         & \emph{chinensis}         & Pak choi, bok choi       & Leaves \\
                         & \emph{pekinensis}        & Chinese cabbage, pe-tsai & Leaves \\
                         & \emph{parachinensis}     & Choy sum                 & Leaves \\
                         & \emph{ruvo}              & Broccoleto               & Shoots \\
                         & \emph{perviridis}        & Komatsuna, Tendergreen   & Leaves \\
\emph{Brassica napus}    & \emph{pabularia}         & Leaf rape, nabicol       & Leaves \\
                         & \emph{napobrassica}      & Swede                    & Roots \\
\emph{Brassica juncea}   & \emph{rugosa}            & Mustard greens           & Leaves \\
                         & \emph{capitata}          & Head mustard             & Heads \\
                         & \emph{crispifolia}       & Cut leaf mustard         & Leaves \\
\bottomrule
\end{tabular}%
}
\caption{\textbf{Main \emph{Brassica} crops, their common names, and the part of the plant that is consumed.} Table obtained from Cartea et al.[TODO]}
\label{intro:table1xx:brassicas}
\end{table}

Aside from being an economically important crop, the *Brassica* species are also a model for gene retention.
The genomes of *Brassica* species have undergone genome duplication events relative to *Arabidopsis* since the two genera diverved 43 million years ago[@beilstein_dated_2010].
There is evidence for an ancestor of the *Brassica* lineage being a hexaploid, with estimates of when the genome triplication occurred varying from 7.9 - 14.6 million years ago[@lysak_chromosome_2005] and 23 million years ago[@beilstein_dated_2010].
Subsequent diploidization of this hexaploid ancestor has given us the diploid *Brassica* species we have today.
*B. rapa* and *B. oleracea* diverged 0.12 - 3.7 million years ago[@cheung_comparative_2009; @inaba_phylogenetic_2002], with the process of chromosome rearrangement resulting in a chromosome number of 10 for *B. rapa* (A genome)[@rapa_genome_2011] and 9 for *B. oleracea* (C genome)[@oleracea_genome_2014].
It is thought that the interspecies hybidization event resulting in the allopolyploid *B. napus* occurred less than 10,000 years ago[@rana_conservation_2004].
Both the ancient hexaploid state of the *Brassica* genomes and the interspecies hybridization event mean that *B. napus* has a greatly increased gene number than Arabidopsis (101,040[@napus_genome_2014] relative to 25,498[@the_arabidopsis_genome_initiative_analysis_2000]), with genes in Arabidopsis often having multiple homologues in the *B. napus* genome.
Despite large scale genome rearrangements, extensive collinearity between the *Brassica* and Arabidopsis genomes remains[@rapa_genome_2011; @oleracea_genome_2014; @napus_genome_2014].
This genomic collinearity and relatedness of the two plant species has been exploited to translate research from the model plant to the crop species, as well as investigate the effects of gene duplication.

### How does flowering time affect the cultivation of *Brassica* species?

The success of many *Brassica* crops is dependent on the flowering time of the crop.
The edible componant of both broccoli (*B. rapa* var. *botrytis italica*) and cauliflower (*B. rapa* var. *botrytis botrytis*) are the plant inflorescences, and the timing of the floral transition and floral development in general is very important for these crops as a consequence.
Using variation in curd formation in cauliflower, a number of potential candidate genes were identified as controlling the response to temperature, with some of these genes being homologues of floral genes in Arabidopsis[@matschegewski_genetic_2015].
With other *Brassica* crops, such as Chinese cabbage (*B. rapa* var. *pekinensis*) the prevention of flowering is desired.
Chinese cabbage is grown for its leaves (Table \ref{intro:table1xx:brassicas}).
If the plant transitions to floral growth it will bolt, significantly reducing its economic value.
The expression of a floral repressor, a *B. rapa* homologue of *FLC*, was found to correlate with bolting time in different Chinese cabbage lines[@li_molecular_2005].

*B. napus* crops are predominantly used as oilseed crops, in which the timing of the floral transition impacts both when the seed filling period begins and how long it progresses.
Indeed, the interconnected nature of yield and flowering has been suggested by association studies finding regions of the genome associated with both traits[@schiessl_diverse_2015; @shi_unraveling_2009].
The yield of oilseed rape crops is determined by the number of seeds the plants produce per area and the weight of each seed.
Numbers of pods and seeds are largely determined during a 3 week phase after flowers have formed[@mendham_effects_1981; @ahdb_oilseed_rape], with the quality of the seed dependent on a period of seed filling.
The seed quality is related to temperature, with cooler conditions extending seed filling, and the rate of photosynthesis, with the majority of oil in the seed accumulated during the second half of seed filling[@ahdb_oilseed_rape].
The effect of photosynthesis during the seed filling period is potentially of greater significance in *B. napus* relative to other crops as the remobilization of carbohydrates accumulated before flowering is ~12%, compared to 20-50% in wheat[@habekotte_quantitative_1993; @ahdb_oilseed_rape].
Yield of winter oilseed rape has been found to be related to the size of the crop at flowering[@mendham_effects_1981].
This is in turn a function of the length of time between the beginning of spring, when mean growing temperatures exceeded 5 &deg;C, and when the plants flowered in late May.
Therefore, the highest yielding years were those where spring was early and flowering late, allowing the longest period of time for growth in this critical period.
Similar findings came out of modelling the growth of *B. napus*, with higher yields predicted to be obtained by delaying plant maturity and promoting earlier flowering, to ensure the seed filling period is as long as possible[@habekotte_options_1997]
Therefore, when flowering occurs during the growing season, and how that relates to the climate in which the crop is grown, can heavily influence the yield and quality of the crop.

Whether the oilseed rape is a spring or winter variety is also important, as different growing regions require different types of crop.
In Europe and Asia, winter oilseed rape is predominantly grown, whereas in Australia, Canada, and northern Europe spring types are generally grown[@friedt_oilseed_2009].
For Canada and northern Europe, the requirement for spring types results form harsh winters that prevent the crop from being overwintered.
Therefore, the vernalization requirement of a variety is important to consider for the planned crop rotation a particular farmer or growing region requires.
Additionally, the length and severity of cold required by a variety will dictate whether that variety is suitable to a particular application.

Finally, the availability of pollinators can significantly impact the yield of *B. napus* crops.
Preventing pollinators visiting winter oilseed rape plants led to a 27% decrease in the number of seeds produced and a 30% decrease in the seed weight per pod[@stanley_pollinators_2013].
In addition the diversity of those pollinators visiting the plants is related to oilseed rape yield[@zou_wild_2017]
Changes to flowering time will affect the pollinators that are available to the flowering plants.
This has been found to profoundly affect the reproductive success of perennial wildflowers[@rafferty_pollinator_2012].
Therefore, as the yield of oilseed rape is influenced by pollinator availability[@stanley_pollinators_2013; @zou_wild_2017], the correct timing of flowering is required.

### Work on the control of flowering in Brassica species {#section:intro:brassicafloweringgenes}

Extensive work on how the floral response is controlled in Arabidopsis has facilitated understanding of floral control in a range of crop plants[@jung_flowering_2009].
Homologues of the floral genes (section \ref{section:intro:arabidopsis}) have been detected in the genomes of *Brassica* species, which due to the gene multiplication events that have occurred in the Brassica lineage are often present as multiple copies[@schiessl_capturing_2014].
However, identification of whether these *B. napus* homologues have similar functions to their counterparts in Arabidopsis, and of functional differences between the homologues, is often lacking.

Likely as a result of both spring and winter varieties of *Brassica* crops being of such economic value, the vernalization pathway has arguably been the most well studied flowering pathway in Brassicas.
Association studies focussing on mapping the vernalization response in *B. rapa*[@zhao_brflc2_2010; @kole_evidence_2001; @osborn_comparison_1997; @lou_quantitative_2007; @axelsson_multiple_2001; @schranz_characterization_2002], *B. oleracea*[@axelsson_multiple_2001; @okazaki_mapping_2007; @ridge_role_2015; @irwin_nucleotide_2016], and *B. napus*[@osborn_comparison_1997; @zou_comparative_2012; @wang_flowering_2011; @long_flowering_2007] have identified regions containing homologues of *FLC* and *FRI* as explaining flowering time variation.
These homologues exhibit similar decreases in expression during cold as their Arabidopsis counterpart[@li_molecular_2005; @lin_differential_2005] and have been investigated to determine if they have diverged in function or not.
Expression of five different *FLC* homologues in Arabidopsis conferred a vernalization requirement in a rapid-cycling accession of Arabidopsis[@tadege_control_2001].
Interestingly, the delay in flowering as a result of the transgenic gene varied depending on the homologue, suggesting that the genes have diverged roles in *B. napus*[@tadege_control_2001].
The results from association studies carried out with different mapping populations have also suggested that the *FLC* copies in *Brassicas* have diverged, with the copies on chromosomes A10 and A2 having stronger associations with flowering time[@zou_comparative_2012; @schranz_characterization_2002].
Similarly, multiple *FLC* homologues from *B. rapa* delayed flowering when overexpressed in both Arabidopsis and Chinese cabbage, suggesting a conservation of function[@kim_delayed_2007].
*FRI* homologues from *B. oleracea* were able to complement an Arabidopsis accession that contains a nonfunctional copy of the gene[@irwin_functional_2012], indicating conservation of function.
Despite all homologues being able to complement Arabidopsis, structure of the *FRI* homologues from *B. oleracea* have diverged with alterations in the number of coiled-coil domains, potentially impacting protein-protein interactions[@irwin_functional_2012].
Therefore, although it has been established that *FLC* and *FRI* seem to be important in the vernalization pathways of both *Brassica* crops and Arabidopsis[@shindo_role_2005], how the copies of these genes have diverged in *Brassica* is only beginning to be understood.

Genes in other flowering time pathways, and the floral integrators, have also been investigated in *Brassica* species.
Genes involved with the circadian clock have been retained in the *B. rapa* genome, suggesting that the dosage of the genes is important for their function[@lou_preferential_2012].
In particular, homologues of the clock sensitive gene *CO* are associated with changes in flowering time in both *B. oleracea*[@bohuon_association_1998] and *B. nigra*[@lagercrantz_comparative_1996; @osterberg_naturally_2002].
Homologues of *TFL1* were identified in *B. rapa*, *B. oleracea*, and *B. napus*, with expression in the flower in the latter species in line with expression of the gene in Arabidopsis[@mimida_terminal_1999].
Mutations in the A10 copy of *TFL1* in *B. napus* caused a delay in flowering, affected internode elongation, and resulted in an increase in seed number and weight[@guo_mutations_2014].
Homologues of *FT* in *B. napus* exhibited different expression patterns, with certain copies having a stronger effect on flowering than others[@guo_mutations_2014].
Expression differences were observed between the homologues of *FT* in *B. rapa*, *B. oleracea*, and *B. napus*[@wang_promoter_2012].
Within *B. napus*, one copy was silenced as a result of transposon insertion into the promoter region and the expression of another two copies was crop type specific[@wang_promoter_2012].
Transposon mediated changes to the expression of an *FT* homologues were also identified in *B. rapa*, resulting in flowering time differences.
This suggested that this copy of *FT* has retained a function similar to its counterpart in Arabidopsis[@zhang_transposon_2015].
Arrest of floral development is required in broccoli and cauliflower to form the heads correctly.
Interestingly, the floral genes predicted to cause the arrest (*LFY*, *AP1*, and *TFL1*) were not implicated, causing the authors to suggest other floral meristem genes are mediating the change relative to Arabidopsis[@duclos_meristem_2008].
Links between flowering time and *SOC1* homologues have been identified in *B. rapa*[@franks_variation_2015], with expression differences detected between the different homolgoues in *B. rapa* and *B. juncea*[@franks_variation_2015; @sri_sequence_2015]

Despite evidence of of flowering time genes homologues having similar roles in *Brassica* species, in depth analysis of how different homologues are behaving is often lacking.
This is not the case for all genes however, with the roles of *FT*, *FLC*, and *FRI* homologues in *Brassica* species being dissected in a copy specific manner.
These investigations have revealed that individual copies have indeed diverged in function and behaviour.
